<!DOCTYPE html>
<html lang="" dir="ltr">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>The coevolutionary mosaic of bat betacoronavirus emergence risk</title>
    <script src="https://hypothes.is/embed.js" async></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
    integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />
    <style>
        @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,400;0,500;1,400;1,500&display=swap');

        :root {
            --citation: #1976D2;
            --structure: hsl(210, 61%, 43%);
            
            --light: #474747;
            --pale: #a5a5a5;
            --grey: #e0e0e0;
            --nav: #fdfdfd;

            --width: 820px;
            --inner-space: 1.2em;
        }

        body {
            font-family: "Roboto", sans-serif;
            margin: 0 0;
            padding: 0 0;
        }

        article > p {
            line-height: 135%;
        }
        article > h1 {
            padding-top: 3em;
        }
        a {
            text-decoration: none;
            color: #000;
            border-bottom: 1px solid var(--structure);
        }
        a:hover {
            font-weight: 500;
        }
        .citation {
            color: var(--citation);
        }
        .container {
            margin: 0em auto;
            max-width: var(--width);
            padding: var(--inner-space);
        }
        header > .container {
            margin-top: 4em;
        }
        code, pre {
            font-family: "Roboto Mono", monospace;
        }
        pre {
            white-space: pre-wrap;
        }
        h1, h2, h3 {
            color: var(--structure);
            font-family: "Roboto Condensed", sans-serif;
            font-weight: 400;
        }
        header {
            font-family: "Roboto Condensed", sans-serif;
            z-index: 5;
            color: black;
            margin-top: 4rem;
        }
        .header--title {
            font-weight: 400;
            font-size: 2rem;
            margin-bottom: 1rem;
        }
        .header--authors > span {
            color: var(--light);
            font-weight: 400;
        }
        main {
            padding-top: 8em !important;
            padding-bottom: 8em !important;
        }
        footer {
            background-color: var(--grey);
            font-weight: 300;
        }
        .abstract {
            color: var(--light);
            margin-bottom: 5em;
        }
        figure {
            max-width: 100%;
            border: 1px solid var(--pale);
            border-radius: 2px;
            margin: 1em 0.5em;
        }
        figure > img {
            max-width: 100%;
            display: block;
            margin: 0em auto;
        }
        figure > figcaption {
            padding: var(--inner-space);
        }
        table {
            width: 100%;
            min-width: 100%;
            border-collapse: collapse;
        }
        table > caption {
            padding: var(--inner-space);
        }
        figcaption, caption {
            color: var(--light);
            font-weight: 300;
        }
        thead {
            color: var(--structure);
        }
        td, th {
            padding: 3px;
        }
        tr {
            border-top: 1px solid var(--structure);
            border-bottom: 1px solid var(--structure);
        }
        tbody > tr:nth-child(even) {background: var(--grey)};
        tbody {
            border-bottom: 1px solid var(--structure);
        }
        #refs {
            font-size: 90%;
            column-count: 2;
        }
        #refs > .csl-entry {
            padding-bottom: 0.2em;
            padding-top: 0.2em;
        }
        .hanging-indent > .csl-entry {
            padding-left: 15px ;
            text-indent: -15px ;
        }    </style>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/katex.min.css"
    integrity="sha384-D+9gmBxUQogRLqvARvNLmA9hS2x//eK1FhVb9PiU86gmcrBrJAQT8okdJ4LMp2uv" crossorigin="anonymous" />
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/katex.min.js"
    integrity="sha384-483A6DwYfKeDa0Q52fJmxFXkcPCFfnXMoXblOkJ4JcA8zATN6Tm78UNL72AKk+0O"
    crossorigin="anonymous"></script>
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/contrib/auto-render.min.js"
    integrity="sha384-yACMu8JWxKzSp/C1YV86pzGiQ/l1YUfE8oPuahJQxzehAjEt2GiQuy/BIvl9KyeF"
    crossorigin="anonymous"></script>
    <script>
        document.addEventListener("DOMContentLoaded", function () {
            renderMathInElement(document.body, {
                delimiters: [
                { left: "$$", right: "$$", display: true },
                { left: "$", right: "$", display: false }
                ]
            });
        });
    </script>
</head>

<body>
    <header>
        <div class="container">
            <div class="header--title">The coevolutionary mosaic of bat betacoronavirus emergence risk</div>
            <div class="header--authors">
                <span class="header--author">
                Norma R Forero-Muñoz
                <sup class="affilcode">1</sup>
                <sup class="affilcode">2</sup>
                <sup class="affilcode">‡</sup>
                </span><span class="header--author">
                Renata L. Muylaert
                <sup class="affilcode">3</sup>
                </span><span class="header--author">
                Stephanie N. Seifert
                <sup class="affilcode">4</sup>
                </span><span class="header--author">
                Gregory F. Albery
                <sup class="affilcode">5</sup>
                </span><span class="header--author">
                Daniel J. Becker
                <sup class="affilcode">6</sup>
                </span><span class="header--author">
                Colin J. Carlson
                <sup class="affilcode">7</sup>
                <sup class="affilcode">8</sup>
                <sup class="affilcode">9</sup>
                <sup class="affilcode">‡</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-0735-5184" target="_blank">
                    Timothée Poisot
                </a>
                <sup class="affilcode">1</sup>
                <sup class="affilcode">2</sup>
                <sup class="affilcode">‡</sup>
                </span>
            </div>
        </div>
    </header>
    
    <main class="container">
        <section class="abstract">
            Pathogen evolution is one of the least predictable components of disease emergence, particularly in nature. Here, building on principles established by the geographic mosaic theory of coevolution, we develop a quantitative, spatially-explicit framework for mapping the evolutionary risk of viral emergence. Driven by interest in diseases like SARS, MERS, and COVID-19, we examine the global biogeography of bat-origin betacoronaviruses, and find that coevolutionary principles suggest geographies of risk that are distinct from the hotspots and coldspots of host richness. Further, our framework helps explain patterns like a unique pool of merbecoviruses in the Neotropics, a recently-discovered lineage of divergent nobecoviruses in Madagascar, and–most importantly–hotspots of diversification in southeast Asia, sub-Saharan Africa, and the Middle East that correspond to the site of previous zoonotic emergence events. Our framework may help identify hotspots of future risk that have also been previously overlooked, like west Africa and the Indian subcontinent, and may more broadly help researchers understand how host ecology shapes the evolution and diversity of pandemic threats.


            <div class="artifacts">
                <a href="draft.pdf" title="Draft PDF" target="_blank">draft</a>
                <a href="preprint.pdf" title="Preprint PDF" target="_blank">preprint</a>
            </div>
        </section>
        
        <article>
            <p>Disease emergence is complex, and is driven not only by animal-human contact, but also by the underlying evolutionary dynamics in viral reservoirs.<span class="citation" data-cites="Plowright2017Pathways"><sup>1</sup></span> Although host richness is often used as a superficial proxy for spillover risk,<span class="citation" data-cites="Anthony2017GloPat Ruiz-Aravena2022EcoEvo Sanchez2022Strategy"><sup><strong>Anthony2017GloPat?</strong>,<strong>Ruiz-Aravena2022EcoEvo?</strong>,<strong>Sanchez2022Strategy?</strong></sup></span> these approaches oversimplify the relevant interspecific heterogeneity in immunology, behavior, and other traits, and therefore overlook unique host pools that allow for the rapid evolution of highly divergent viruses.<span class="citation" data-cites="Agosta2010How"><sup>2</sup></span> In the case of generalist pathogens like betacoronaviruses, there is conceptual and empirical support to the idea that these community-level mechanisms are even more important,<span class="citation" data-cites="Power2004Pathogen"><sup>3</sup></span> particularly given that cross-species transmission may, as a rule, structure viral evolution more than co-divergence with hosts.<span class="citation" data-cites="Geoghegan2017Comparative"><sup>4</sup></span> This creates a disconnect between coevolutionary theory and most existing ecological frameworks for mapping spillover risk.</p>
            <p>The geographic mosaic theory of coevolution (GMTC) attempts to explicitly connect microevolutionary dynamics to the macroecology and biogeography of symbiotic interactions.<span class="citation" data-cites="Thompson2005Geographic"><sup>5</sup></span> The GMTC posits that coevolutionary processes among pairs<span class="citation" data-cites="Thompson1994Coevolutionary"><sup>6</sup></span> or complexes<span class="citation" data-cites="Janzen1980When"><sup>7</sup></span> of species are structured in space by the rippling effects of abiotic conditions onto evolutionary mechanisms, giving rise to fragmented systems with different ecologies over large spatial extents.<span class="citation" data-cites="Price2002Macroevolutionary"><sup>8</sup></span> The GMTC predicts a spatial fragmentation of coevolutionary dynamics under the joint action of three processes:<span class="citation" data-cites="Gomulkiewicz2007Dos"><sup>9</sup></span> coevolutionary hot- and coldspots, which appear when the intensity of <em>interaction</em> (in terms of reciprocal fitness consequences) varies spatially; selection mosaics, wherein the intensity of <em>selection</em> varies across space, driven by both the biotic complexity of the community (locally diverse hosts and viruses are more biotically complex) and the local favorability of the environment;<span class="citation" data-cites="Thrall2007Coevolution"><sup>10</sup></span> and trait remixing, which occurs when coevolutionary dynamics change when community-level <em>functional traits</em> change through meta-community dynamics.</p>
            <p>Here, we apply the GMTC to explore and explain the global biogeography of betacoronaviruses, the group that includes SARS-CoV, MERS-CoV, and SARS-CoV-2. In their bat reservoirs, coronaviruses evolve through a mix of host jumps, recombination among disparate lineages, and, to a lesser degree, co-divergence with their hosts—<span class="citation" data-cites="Anthony2017Global"><sup>11</sup></span>a mix of mechanisms that creates a complex and nonlinear relationship between host diversity and viral emergence. Working from a recently published database of bat hosts of betacoronaviruses, we test whether spatial structure in bat-betacoronavirus coevolution is identifiable at a global scale. Aiming to explain these patterns, we develop a generalized framework for applying the GMTC to host-virus interactions, with a specific emphasis on the potential to create independent coevolutionary dynamics (and therefore spatial fragmentation in risk) through heterogeneity. We develop a trivariate risk assessment system that connects each GMTC mechanism to a quantifiable aspect of host-virus interactions: (i) viral sharing rates in host communities, representing the strength of potential interaction between viruses and any one host (i.e., places where viruses undergo constant host switching may be coevolutionary coldspots); (ii) the phylogenetic diversity of hosts, as a proxy for variation in the immunological mechanisms that antagonize viruses (i.e., the selection mosaic); and (iii) the local uniqueness of the bat community, representing the potential for viruses to be exposed to novel host traits (e.g., variation in receptor sequences). Together, we argue that these can be used to identify and map the evolutionary drivers that—in conjunction with transmission processes (e.g., viral prevalence in reservoirs and animal-human contact rates)— determine disease emergence risk.</p>
            <h1 id="results-and-discussion">Results and Discussion</h1>
            <h2 id="bat-and-betacoronavirus-biogeography-are-broadly-consistent">Bat and betacoronavirus biogeography are broadly consistent</h2>
            <p>Most previous work has assumed that the presence or richness of key groups of bat hosts are predictive of coronavirus diversity.<span class="citation" data-cites="Anthony2017GloPat Ruiz-Aravena2022EcoEvo"><sup><strong>Anthony2017GloPat?</strong>,<strong>Ruiz-Aravena2022EcoEvo?</strong></sup></span> Projecting bat and betacoronavirus phylogeny over space (fig. 1), we find support for the idea that bat community assembly is directly responsible for a global mosaic of viral evolution. The distinct groupings (represented by different colors, symbolizing positions in a subspace formed by the first two phylogenetic principal components) are essentially equivalent between the two groups, and can be coarsely delineated as (1) south and southeast Asia; (2) east Asia (including northern China), west Asia, and the Mediterranean coast; (3) Eurasia above a northing of 40; and (4) Africa and Latin America. In some cases, this diverges from expectations about coronavirus biogeography: for example, previous work has rarely flagged India as a region of interest, but for both bats and betacoronaviruses, the subcontinent falls into the same regions as the southeast Asian peninsula (and indeed, the region is home to known bat hosts of multiple betacoronavirus subgenera, including nobecoviruses, sarbecoviruses, and merbecoviruses).<span class="citation" data-cites="Ruiz-Aravena2022Ecology"><sup>12</sup></span></p>
            <figure>
            <img src="figures/combined_biogeo.png" id="fig:biogeo" alt="Figure 1: Bat and betacoronavirus biogeographic regions. Phylogeography of bats (top) and viruses (bottom) is categorized based on an analysis of bat distributions, paired with bat or virus phylogeny. The different colors show tendencies to separate alongside the first two components of a PCoA. Note that the PCoA for the bats and viruses are independent, and so cannot be compared directly – that being said, the fact that different regions cluster in the same way across maps be directly compared." /><figcaption aria-hidden="true">Figure 1: <strong>Bat and betacoronavirus biogeographic regions.</strong> Phylogeography of bats (top) and viruses (bottom) is categorized based on an analysis of bat distributions, paired with bat or virus phylogeny. The different colors show tendencies to separate alongside the first two components of a PCoA. Note that the PCoA for the bats and viruses are independent, and so cannot be compared directly – that being said, the fact that different regions cluster in the same way across maps be directly compared.</figcaption>
            </figure>
            <p>Overall, these results suggest that the boundaries of bat and betacoronavirus biogeographic regions are broadly consistent at a global scale; perfect matching between the biogeographic regions would have indicated that the signal of virus distribution is fully predicted by bat hosts ranges. Areas for which the biogeographic regions for bats and betacoronaviruses differ are primarily (i) southeast Asia and southern China, and (ii) the Arabian peninsula, which are both regions where zoonotic transmission has been documented (potentially driving a unique level of viral sampling effort that generates these patterns). These spatially limited mismatches nonwithstanding, the large level of congruence may be surprising, given that cross-species transmission may play a stronger role in coronavirus diversification than cospeciation—<span class="citation" data-cites="Anthony2017Global"><sup>11</sup></span>a property that would theoretically allow for substantial broad divergence in their biogeography. However, host jumps at the family level or higher are relatively rare and significant events in coronavirus evolutionary history;<span class="citation" data-cites="Anthony2017Global Latinne2020Origin"><sup>11,13</sup></span> as a result, the mosaic of betacoronavirus phylogeography is assembled from a set of overlapping smaller coevolutionary systems, superimposed in space and filtered by the importance of different subgroups in local host communities. For example, the most speciose and cosmopolitan family of bats, the vesper bats (Vespertilionidae), are considered the primary hosts of the subgenus <em>Merbecovirus</em> (MERS-like viruses);<span class="citation" data-cites="Latinne2020Origin Ruiz-Aravena2022Ecology"><sup>12,13</sup></span> but in the Americas, where merbecoviruses are the only lineage present, they have only been found in other bat taxa (<em>e.g.</em>, Molossidae, Phyllostomidae).<span class="citation" data-cites="Anthony2013Coronaviruses Goes2013Novel Goes2016Genetic Brandao2008Coronavirus"><sup>14–17</sup></span> At the coarsest scale, these heterogeneities are lost, and betacoronavirus biogeography tracks the deep rifts in bat evolutionary history—but within broad regions, the component coevolutionary systems may have very different dynamics.</p>
            <h2 id="hotspots-of-bat-and-betacoronavirus-biodiversity-are-distinct">Hotspots of bat and betacoronavirus biodiversity are distinct</h2>
            <p>Bats, the second most diverse groups of mammals, are found worldwide; gradients in their species richness generally track broader patterns of mammal diversity,<span class="citation" data-cites="Tanalgo2022Mapping"><sup><strong>Tanalgo2022Mapping?</strong></sup></span> with a striking Neotropical hotspot (especially in the Amazon basin) and a secondary hotspot centered in Indochina. These hotspots of bat diversity are generally presumed to be hotspots of viral adaptive radiation, and therefore areas of concern for human health.<span class="citation" data-cites="Anthony2017GloPat Olival2017HosVir"><sup><strong>Anthony2017GloPat?</strong>,<strong>Olival2017HosVir?</strong></sup></span> However, the hotspots of known bat betacoronavirus hosts show a distinct pattern, with primary hotspots (both in terms of area and higher values) of host richness situated in southeast Asia, parts of southern Europe, and to a lesser extent parts of Africa in the -25-0 range of latitudes (fig. 2; top). Although hundreds of species likely host undiscovered betacoronaviruses, machine learning predictions have suggested that these undiscovered reservoirs should follow the same diversity gradient.<span class="citation" data-cites="Becker2022OptPre"><sup><strong>Becker2022OptPre?</strong></sup></span> In principle, these hotspots of locally-diverse, virus-rich bat communities should drive more adaptive diversification in their viruses.</p>
            <figure>
            <img src="figures/combined_richness.png" id="fig:richness" alt="Figure 2: Bat and betacoronavirus diversity. Top panel: diversity of known bat hosts of betacoronaviruses in our dataset. This map shows that the region with the largest number of possible hosts is South-Eastern Asia. Bottom panel: congruence between the evolutionary distinctiveness of the hosts (grey to blue) and the viruses (grey to red). Darker areas have higher combined evolutionary distinctiveness for the entire bat-virus system." /><figcaption aria-hidden="true">Figure 2: <strong>Bat and betacoronavirus diversity.</strong> Top panel: diversity of known bat hosts of betacoronaviruses in our dataset. This map shows that the region with the largest number of possible hosts is South-Eastern Asia. Bottom panel: congruence between the <em>evolutionary</em> distinctiveness of the hosts (grey to blue) and the viruses (grey to red). Darker areas have higher combined evolutionary distinctiveness for the entire bat-virus system.</figcaption>
            </figure>
            <p>However, we find that the global pattern of betacoronavirus phylogenetic distinctiveness is quite distinct from both bat host richness and phylogenetic distinctiveness (fig. 2; bottom). In contrast to the sparsity of Neotropical betacoronavirus hosts, South and Central America have the most evolutionary distinct hosts <em>and</em> viruses, followed by secondary hotspots in southeast Asia and the Rift Valley region have mostly distinct viruses. Some degree of sampling bias may contribute to these patterns: for example, the Neotropics are one of the places where the fewest bat betacoronavirus sequences have been generated,<span class="citation" data-cites="Worobey2022HuaMar Temmam2022BatCor Boni2020EvoOri"><sup><strong>Worobey2022HuaMar?</strong>,<strong>Temmam2022BatCor?</strong>,<strong>Boni2020EvoOri?</strong></sup></span> resulting in a sparser phylogenetic tree, and artificially inflating distinctiveness; conversely, disproportionate research effort in eastern China<span class="citation" data-cites="Cohen2022SamStr"><sup><strong>Cohen2022SamStr?</strong></sup></span> may have led to a more complete inventory of the local diversity of coronaviruses, again inflating these metrics relative to underlying patterns. Even accounting for these potential biases, though, there is obvious heterogeneity in betacoronavirus evolutionary distinctiveness that is distinct from overall bat diversity.</p>
            <p>Overall, these patterns recapitulate the evolutionary history of both the order Chiroptera and the genus <em>Betacoronavirus</em>. Horseshoe bats (Rhinolophidae) include the reservoirs of the SARS-like viruses (subgenus <em>Sarbecovirus</em>), the group of pandemic threats that have been of the greatest interest to researchers<span class="citation" data-cites="Latinne2020OriCro"><sup><strong>Latinne2020OriCro?</strong></sup></span> (and so have been sampled most intensively).<span class="citation" data-cites="Cohen2022SamStr"><sup><strong>Cohen2022SamStr?</strong></sup></span> The hotspots of host richness and viral diversity in southeast Asia—both of which are disproportionately high, considering the global landscape of bat species richness—are almost entirely driven by viral adaptive radiation through host switching within this clade<span class="citation" data-cites="Becker2022OptPre Ruiz-Aravena2022EcoEvo"><sup><strong>Becker2022OptPre?</strong>,<strong>Ruiz-Aravena2022EcoEvo?</strong></sup></span>. In contrast, the Neotropical hotspot of viral distinctiveness is driven by isolation by host vicariance. Out of the four main groups of betacoronaviruses, only merbecoviruses have been found in animals in the Americas— an introduction that is generally presumed to be ancient.<span class="citation" data-cites="Ruiz-Aravena2022EcoEvo Olival2020PosRev"><sup><strong>Ruiz-Aravena2022EcoEvo?</strong>,<strong>Olival2020PosRev?</strong></sup></span> While comparatively understudied, New World merbecoviruses have been found in the ghost-faced bats (Mormoopidae), Neotropical leaf-nosed bats (Phyllostomidae), and free-tailed bats (Molossidae).<span class="citation" data-cites="Anthony2013CorBat Goes2013NovBat Goes2016GenDiv Brandao2008CorDet"><sup><strong>Anthony2013CorBat?</strong>,<strong>Goes2013NovBat?</strong>,<strong>Goes2016GenDiv?</strong>,<strong>Brandao2008CorDet?</strong></sup></span> The former two groups and a clade of the latter are endemic to the Neotropics, while the explosive adaptive radiations of the phyllostomids are responsible for the hotspot of bat diversity in the Amazon.<span class="citation" data-cites="Ammerman2012FirMol"><sup><strong>Ammerman2012FirMol?</strong></sup></span> Together, these clades of New World bats play host to a distinct regime of betacoronavirus coevolution.</p>
            <p>Our approach is potentially limited by sampling bias: key hotspots identified by our model have, indeed, been sampled intensely following major zoonotic emergence events. In these areas, more betacoronavirus hosts will have been discovered, leading to higher overall diversity and potentially higher sharing. Similarly, hotspots of evolutionary uniqueness - as in the Arabian peninsula - could reflect much broader lineages that have only been sampled in focal areas for public health. While the discovery of new branches of bat-betacoronavirus coevolution is certainly likely, and might change some of the observed patterns, our framework is likely to be fairly robust: the 126 hosts in our study capture nearly 10% of global bat diversity, and the underlying evolutionary patterns they represent are much less sensitive to new information than any inferences about viral evolution.</p>
            <h2 id="coevolutionary-regimes-structure-evolutionary-potential-for-zoonotic-emergence">Coevolutionary regimes structure evolutionary potential for zoonotic emergence</h2>
            <p>The existence of well-defined cophylogenetic regions suggests that the bat-betacoronavirus system is spatially fragmented enough to create divergent coevolutionary trajectories; in turn, this coevolutionary mosaic may alter the risk of zoonotic emergence. These ideas are, respectively, supported by the existence of hotspots of viral uniqueness and the diverse origins of human betacoronaviruses. Together, this framework points to a predictable relationship between host community structure and coevolutionary pressure: phylogeographic structure in bat hosts (and their diverse immune strategies)<span class="citation" data-cites="Banerjee2020NovIns"><sup><strong>Banerjee2020NovIns?</strong></sup></span> creates a landscape of selective pressure; the trajectory of viruses’ coevolutionary response is, in turn, constrained by their opportunities for either specialization or diversification through host jumps and recombination.</p>
            <p>Based on the geographic mosaic theory of coevolution, we developed a trivariate map of coevolutionary pressure (fig. 3): (1) <em>host phylogenetic diversity</em>: a high diversity of evolutionary histories should expose viruses to more variation in host immune traits; (2) <em>host community uniqueness</em>: exposure to greater host trait heterogeneity can drive viral diversification, and coevolving with more unique host communities should create more unique branches of viral evolution; and (3) propensity for <em>viral sharing</em>: frequent cross-species transmission may act as a buffer on selective pressure, while lower rates of exchange may enable more simultaneous trajectories of viral specialization to coexist within a given community. We combine global maps of all three to generate a map of coevolutionary regimes, where close colors represent similar risks, and paler pixels represent overall higher risk (see Methods). We find that these regions do not neatly overlap with those defined in fig. 1 or fig. 2, reinforcing the notion that local-scale coevolutionary mosaics can form within cophylogenetic regions.</p>
            <figure>
            <img src="figures/risk_trivariate.png" id="fig:trivariate" alt="Figure 3: Trivariate additive mapping of the components of risk. Viral sharing runs from yellow (low) to blue (high); host phylogenetic diversity runs from pink (low) to high (green); and host compositional uniqueness runs from cyan (low) to red (high). The GMTC suggests that the highest evolutionary potential for emergence exists in unique and diverse host communities with low viral sharing, i.e. pixels around yellow. All components within bat host ranges are scaled in brightness so that a pixel with no sharing, no phylogenetic diversity, and no compositional uniqueness would be black, and a pixel with maximal values for each would be white. The individual layers that compose this figure are given in supplementary material." /><figcaption aria-hidden="true">Figure 3: <strong>Trivariate additive mapping of the components of risk.</strong> Viral sharing runs from yellow (low) to blue (high); host phylogenetic diversity runs from pink (low) to high (green); and host compositional uniqueness runs from cyan (low) to red (high). The GMTC suggests that the highest evolutionary potential for emergence exists in unique and diverse host communities with low viral sharing, <em>i.e.</em> pixels around yellow. All components within bat host ranges are scaled in brightness so that a pixel with no sharing, no phylogenetic diversity, and no compositional uniqueness would be black, and a pixel with maximal values for each would be white. The individual layers that compose this figure are given in supplementary material.</figcaption>
            </figure>
            <p>The greatest evolutionary potential for zoonotic emergence exists where pathogen pools have a high genetic diversity and high propensity for cross-species transmission. In our framework, emergence risk is therefore maximized under higher phylogenetic diversity (viruses are exposed to different host clades), higher host uniqueness (viruses are experiencing novel, heterogeneous host traits combinations), and low to medium viral sharing (host-virus pairs can coevolve independently, but divergent viruses may still have opportunities for recombination). In fig. 3, this corresponds to yellow areas (dynamics dominated by low viral sharing, with equal contributions of selection mosaics and trait remixing; southeast Asia, and the Indian sub-continent), green-yellow areas (dynamics with low viral sharing but dominated by the selection mosaic effect of host diversity; sub-Saharan Africa), and red-yellow areas (dynamics with low viral sharing but dominated by trait remixing in host communities; the Middle East). Translating this axis of variation back into a univariate risk map (fig. 4) highlights that this evolutionary landscape has a striking correspondence to regions where zoonotic betacoronaviruses have previously emerged. Our findings align with predictions regarding the spatial location of cross-species transmission. These locations not only pose a potential risk of viral jumps that could endanger human health but also provide valuable information for monitoring wildlife health. This could guide us to determine where and what measures to implement for effectively monitoring wildlife and human betacoronavirus outbreaks before they escalate to critical levels. Nevertheless, there are actually very few documented cases of emergence events, and similarities could be some degree of coincidental.</p>
            <p>Compared to approaches that map emergence risk based only on the number of known bat hosts of betacoronaviruses, our framework suggests regions where high viral sharing dominates coevolutionary dynamics—such as Latin America, or Eurasia above a northing of 30—would pose less of a relative risk of zoonotic emergence. Nevertheless, areas of high host uniqueness coupled with high viral sharing (red-to-pink in fig. 3) could create hotspots facilitated by viral codivergence. Our framework identifies Madagascar, where most bat species are endemic following evolutionary divergence from sister species in both African and Asian continents,<span class="citation" data-cites="Shi2014DeeDiv"><sup><strong>Shi2014DeeDiv?</strong></sup></span> as one such hotspot; interestingly, a recent study<span class="citation" data-cites="Kettenburg2022FulGen"><sup><strong>Kettenburg2022FulGen?</strong></sup></span> reported a novel and highly divergent lineage of nobecoviruses from Madagascar-endemic pteropid bat species (<em>Pteropus rufus</em> and <em>Rousettus madagascariensis</em>), again supporting the predictive power of the coevolutionary framework.</p>
            <figure>
            <img src="figures/risk_map.png" id="fig:risk" alt="Figure 4: Evolutionary potential for zoonotic emergence of bat-origin betacoronaviruses. Risk is a composite measure of the color value and angular distance to the yellow hue in fig. 3 (see Methods). Darker pixels represent areas where the co-evolutionary mechanisms are likely to introduce a strong risk of emergence." /><figcaption aria-hidden="true">Figure 4: <strong>Evolutionary potential for zoonotic emergence of bat-origin betacoronaviruses.</strong> Risk is a composite measure of the color value and angular distance to the yellow hue in fig. 3 (see Methods). Darker pixels represent areas where the co-evolutionary mechanisms are likely to introduce a strong risk of emergence.</figcaption>
            </figure>
            <h2 id="human-landscapes-filter-the-geography-of-emergence-risk">Human landscapes filter the geography of emergence risk</h2>
            <p>The relationship between the underlying pathogen pool and emergence risk is mediated by both human-wildlife interfaces (the probability of spillover) and opportunities for onward horizontal transmission (the probability that spillovers become epidemics)<span class="citation" data-cites="Plowright2017PatZoo"><sup><strong>Plowright2017PatZoo?</strong></sup></span>. It must be noted that the assesment of risk based on the GMTC mechanisms does not account for human presence; for this reason, it represents “potential” level of risk, which must be re-evaluated in the light of human presence. As a proxy for both, we finally overlaid the risk component from the composite map (see above) with the proportion of built land, as a proxy for a mix of habitat disturbance, potential for bat synanthropy or contact with bridge hosts like livestock,<span class="citation" data-cites="Rulli2021LanCha Cui2019OriEvo"><sup><strong>Rulli2021LanCha?</strong>,<strong>Cui2019OriEvo?</strong></sup></span> and human population density and connectivity<span class="citation" data-cites="Plowright2017PatZoo Muylaert2022PreFut Hassell2017UrbDis"><sup><strong>Plowright2017PatZoo?</strong>,<strong>Muylaert2022PreFut?</strong>,<strong>Hassell2017UrbDis?</strong></sup></span> (fig. 5). Accounting for these factors, most of South America and Europe are at comparatively lower risk, as–although densely populated–settlements tend to be in areas with lower potential risk. Conversely, regions like Malaysia and the northern coast of Australia have a high evolutionary risk component, but should represent a relatively lower effective risk due to low human density. However, southeast Asia, the Indian subcontinent, and scattered hotspots in sub-Saharan Africa are at high risk due to the overlap between human populations and natural opportunities for cross-species transmission of betacoronaviruses.</p>
            <figure>
            <img src="figures/risk_compounded.png" id="fig:compound" alt="Figure 5: Overlap between evolutionary potential and ecological opportunity for zoonotic emergence. Overlap of the percent of each pixel occupied by urbanized structures, representing the degree of settlement, on the spillover risk map (where the risk comes only from wildlife, and ignores multi-hosts chains of transmissions including non-bats hosts). Darker pixels correspond to more risk, in that the GMTC-derived risk of fig. 4 is high and the pixel is densely occupied by human populations." /><figcaption aria-hidden="true">Figure 5: <strong>Overlap between evolutionary potential and ecological opportunity for zoonotic emergence.</strong> Overlap of the percent of each pixel occupied by urbanized structures, representing the degree of settlement, on the spillover risk map (where the risk comes only from wildlife, and ignores multi-hosts chains of transmissions including non-bats hosts). Darker pixels correspond to more risk, in that the GMTC-derived risk of fig. 4 is high <em>and</em> the pixel is densely occupied by human populations.</figcaption>
            </figure>
            <p>Reassuringly, these predictions correspond to the geographic origins of the three bat-origin coronaviruses that have recently emerged in human populations. While available information puts the spillover of SARS-CoV-2 in a live animal market in Wuhan, China, the ultimate origin of the virus is almost certainly in a divergent lineage of sarbecoviruses from Indochina that was poorly characterized prior to the pandemic.<span class="citation" data-cites="Worobey2022HuaMar Temmam2022BatCor Boni2020EvoOri"><sup><strong>Worobey2022HuaMar?</strong>,<strong>Temmam2022BatCor?</strong>,<strong>Boni2020EvoOri?</strong></sup></span> Similarly, the SARS-CoV outbreak began in Guangdong province in 2002, reaching humans through small carnivore bridge hosts, but was eventually traced back to a set of likely progenitor viruses found in cave-dwelling horseshoe bats in Yunnan province;<span class="citation" data-cites="Hu2017DisRic"><sup><strong>Hu2017DisRic?</strong></sup></span> nearby, antibody evidence has indicated human exposure to SARS-like viruses.<span class="citation" data-cites="Wang2018SerEvi"><sup><strong>Wang2018SerEvi?</strong></sup></span> MERS-CoV was first detected in Jordan, but is widespread in camels in East Africa and the Middle East, and may have reached its bridge host decades earlier than originally supposed;<span class="citation" data-cites="Muller2014MerCor"><sup><strong>Muller2014MerCor?</strong></sup></span> as a result, the geography of the original bat-to-camel transmission is still widely regarded as uncertain. All of these are broadly consistent with the risk factors we identify. Notably, India and west Africa are additional hotspots that have yet to experience the emergence of a bat coronavirus into human populations, but may still be at risk—particularly given known gaps in bat surveillance,<span class="citation" data-cites="Cohen2022SamStr"><sup><strong>Cohen2022SamStr?</strong></sup></span> and a dense population in both regions with global connectivity. In any of these regions, surveillance on viral reservoirs can be paired with targeted monitoring of high-risk human populations (i.e., those with regular wildlife contact)<span class="citation" data-cites="Xu2004EpiClu"><sup><strong>Xu2004EpiClu?</strong></sup></span> for maximum impact.</p>
            <h1 id="conclusion">Conclusion</h1>
            <p>Bats emerged around 64 million years ago, and are one of the most diverse mammalian orders, with more than 1,400 estimated species.<span class="citation" data-cites="Peixoto2018SynEco Simmons2020BatSpe"><sup><strong>Peixoto2018SynEco?</strong>,<strong>Simmons2020BatSpe?</strong></sup></span> They exhibit a broad variety of habitat use, behaviour, and feeding strategies, putting them at key positions in the delivery and provisioning of several ecosystem services, tied to important ecosystem-derived benefits to humans.<span class="citation" data-cites="Kasso2013EcoEco"><sup><strong>Kasso2013EcoEco?</strong></sup></span> Over two-thirds of bats are known to be either obligate or facultative insectivores, therefore actively contributing for agricultural pest control,<span class="citation" data-cites="Voigt2016BatAnt Williams-Guillen2008BatLim"><sup><strong>Voigt2016BatAnt?</strong>,<strong>Williams-Guillen2008BatLim?</strong></sup></span> and vectors of pathogens that put a risk on human health;<span class="citation" data-cites="Gonsalves2013MosCon Gonsalves2013MosInf"><sup><strong>Gonsalves2013MosCon?</strong>,<strong>Gonsalves2013MosInf?</strong></sup></span> some other species are essential links in many seed-dispersal networks.<span class="citation" data-cites="Mello2011MisPar"><sup><strong>Mello2011MisPar?</strong></sup></span> However, many of these species face a high risk of extinction, particularly given persecution and killings that sometimes follows from messaging about their role in disease emergence. Areas where bats, viruses, and humans co-occur are not always hotspots of risk for human heath; as such, developing more precise ways to map zoonotic hazards can help bats and humans coexist safely, and support the conservation of these important and unique animals.</p>
            <p>Here, we propose a simple framework with broad explanatory power that helps contextualize discoveries like highly divergent nobecoviruses in Madagascar and the once-neglected adaptive radiation of sarbecoviruses in the Indochinese peninsula. In doing so, it advances ecological theory beyond the current state of the art for global maps of emergence risk. For example, previous studies that have used host richness as a proxy have predicted a high diversity of unsampled bat viruses,<span class="citation" data-cites="Olival2017HosVir"><sup><strong>Olival2017HosVir?</strong></sup></span> bat coronaviruses,<span class="citation" data-cites="Anthony2017GloPat"><sup><strong>Anthony2017GloPat?</strong></sup></span> and even specifically betacoronaviruses<span class="citation" data-cites="Becker2022OptPre"><sup><strong>Becker2022OptPre?</strong></sup></span> in both the Amazon and southeast Asia. While we find that both regions are characterized by unique and diverse communities of both hosts and viruses, our framework is able to identify key differences between the two systems. We find that the merbecovirus complex in Latin America has been a unique branch of evolution separate from the rest of the global pool, but with limited potential for viral diversification— a finding that is supported by previous work indicating a higher rate of codivergence in Latin America.<span class="citation" data-cites="Anthony2017GloPat Caraballo2022CroTra"><sup><strong>Anthony2017GloPat?</strong>,<strong>Caraballo2022CroTra?</strong></sup></span> In contrast, in southeast Asia, host richness and viral distinctiveness are high but sharing is low; this suggests a different type of evolutionary dynamics that could generate high local diversity of viruses through host switching and viral recombination (see <em>e.g.</em>,<span class="citation" data-cites="Latinne2020OriCro"><sup><strong>Latinne2020OriCro?</strong></sup></span> as well as the discovery of recombinant viruses with genetic material from both the SARS-CoV and SARS-CoV-2 branches of the Sarbecovirus lineage).<span class="citation" data-cites="Wu2021ComSur"><sup><strong>Wu2021ComSur?</strong></sup></span>Both of these regions are priority areas for sampling, especially given predictions that they contain many bat hosts of undiscovered betacoronaviruses.<span class="citation" data-cites="Becker2022OptPre Cohen2022SamStr"><sup><strong>Becker2022OptPre?</strong>,<strong>Cohen2022SamStr?</strong></sup></span> However, both the evolutionary and ecological aspects of emergence risk are higher in southeast Asia—a fact that will only become more relevant, as bats track shifting climates and exchange viruses with other species, creating a hotspot of elevated cross-species transmission unique to the region.<span class="citation" data-cites="Carlson2022CliCha Muylaert2022PreFut"><sup><strong>Carlson2022CliCha?</strong>,<strong>Muylaert2022PreFut?</strong></sup></span></p>
            <p>Our trivariate additive mapping of components of risk (fig. 3) aims to elicit the complexity of spatial cross-species transmission risk beyond the mere presence or absence of the pathogen host in a specific location. By considering coevolutionary factors such as viral sharing and host uniqueness, we suggest insights that can aid in identifying potential locations for surveillance of betacoronavirus circulation and assessing the risk of cross-species transmission to other mammals. In communities characterized by diverse but unique host populations, with limited viral sharing between them, we could encounter viruses that specialize in targeting the immune system of specific hosts. This implies a low likelihood of infecting novel hosts but, once locally introduced into a new host (either a new species, or an immunologically naïve population), the specialized virus could spread relatively easily due to encountering little immune resistance (CITE PLOWRIGHT). With the right combination of viral traits, such as low disease-induced mortality or high transmission rate, this could lead to successfully spread within the new host community. However, while high adaptation to a specific host can be advantageous, it may also lead to maladaptation when the pathogen encounters a new unsuitable host, potentially resulting in its extinction.</p>
            <blockquote>
            <p>Plowright RK, Foley P, Field HE, Dobson AP, Foley JE, Eby P, Daszak P. Urban habituation, ecological connectivity and epidemic dampening: the emergence of Hendra virus from flying foxes (Pteropus spp.). Proceedings of the Royal Society B: Biological Sciences. 2011 Dec 22;278(1725):3703-12.</p>
            </blockquote>
            <p>Bats—and the spillover of their viruses—are also sensitive to anthropogenic factors others than climate change, including deforestation and other kinds of habitat loss, increased stress, and greater contact with potential bridge hosts like domesticated species.<span class="citation" data-cites="Alves2018GeoVar Treitler2016EffLoc Rulli2021LanCha Mendenhall2014PreBio"><sup><strong>Alves2018GeoVar?</strong>,<strong>Treitler2016EffLoc?</strong>,<strong>Rulli2021LanCha?</strong>,<strong>Mendenhall2014PreBio?</strong></sup></span> This represents a challenge for both conservation strategies and pandemic prevention,<span class="citation" data-cites="Amman2011InvRol"><sup><strong>Amman2011InvRol?</strong></sup></span> but identifying areas at risk, and protecting the health of bats and ecosystems within those zones, can be a win-win intervention for both.<span class="citation" data-cites="Hopkins2021HowIde Plowright2021LanUse OHHLEP2022OneHea"><sup><strong>Hopkins2021HowIde?</strong>,<strong>Plowright2021LanUse?</strong>,<strong>OHHLEP2022OneHea?</strong></sup></span> As we scale these predictions down in space to finer spatial resolutions to guide public health actions,<span class="citation" data-cites="Muylaert2022Present"><sup>18</sup></span> the incorporation of human activity predictors will become more importyant.<span class="citation" data-cites="Ka-WaiHui2006Reasons"><sup>19</sup></span></p>
            <p><strong>Acknowledgements</strong>: We acknowledge that this study was conducted on land within the traditional unceded territory of the Saint Lawrence Iroquoian, Anishinabewaki, Mohawk, Huron-Wendat, and Omàmiwininiwak nations. This work was supported by funding to the Viral Emergence Research Initiative (VERENA) consortium including NSF BII 2021909 and a grant from Institut de Valorisation des Données (IVADO). This research was enabled in part by support provided by Calcul Québec (www.calculquebec.ca) and Compute Canada (www.computecanada.ca). NF is funded by the NSERC BIOS² CREATE program. TP and NF are funded by the Courtois Foundation. RLM was supported by Bryce Carmine and Anne Carmine (née Percival), through the Massey University Foundation. DJB was supported by the National Institute of General Medical Sciences of the National Institutes of Health (P20GM134973).</p>
            <h1 id="methods">Methods</h1>
            <h2 id="known-betacoronavirus-hosts">Known <em>Betacoronavirus</em> hosts</h2>
            <p>We downloaded the data on bats hosts of <em>Betacoronavirus</em> from <code>https://www.viralemergence.org/betacov</code> on Apr. 2022,<span class="citation" data-cites="Becker2022OptPre"><sup><strong>Becker2022OptPre?</strong></sup></span> and filtered it to “known” hosts (established before the emergence of SARS-CoV-2) and “novel” hosts (confirmed through sampling and competence assays since the initial data collection). The original database was assembled by a combination of data mining and literature surveys, including automated alerts on the “bats” and “coronavirus” keywords to identify novel empirical evidence of bats-betacoronaviruses associations; this yielded a total of 126 known hosts, 47 of which were novel hosts. This host–virus list of interactions was obtained through a comprehensive aggregation of GenBank data as well as systematic literature searches,<span class="citation" data-cites="Becker2022OptPre Cohen2022SamStr"><sup><strong>Becker2022OptPre?</strong>,<strong>Cohen2022SamStr?</strong></sup></span> such that we have high confidence in its fitness for the purpose of inference at a large spatial scale.</p>
            <h2 id="bat-occurrences">Bat occurrences</h2>
            <p>We downloaded the rangemap of every current bat species that was classified as an empirically documented host of <em>Betacoronavirus</em> from the previous step, according to recent IUCN data.<span class="citation" data-cites="IUCN2021IucRed"><sup><strong>IUCN2021IucRed?</strong></sup></span> The IUCN data have been assembled to support wildlife conservation efforts, and therefore we do not expect that they are biased by wildlife disease sampling efforts or priority. The range maps were subsequently rasterized using the <code>rasterize</code> function from <code>GDAL</code><span class="citation" data-cites="RouaultEven2022GdaOgr"><sup><strong>RouaultEven2022GdaOgr?</strong></sup></span> at a resolution of approximately 100kmx100km at the equator. For every pixel in the resulting raster where at least one bat host of <em>Betacoronavirus</em> was present, we extract the species pool (list of all known bat hosts), which was used to calculate the following risk assessment components: bat phylogenetic diversity, bat compositional uniqueness, and predicted viral sharing risk.</p>
            <h2 id="bat-phylogenetic-diversity">Bat phylogenetic diversity</h2>
            <p>For every pixel, we measured Faith’s Phylogenetic Diversity<span class="citation" data-cites="Faith1992ConEva"><sup><strong>Faith1992ConEva?</strong></sup></span> based on a recent synthetic tree with robust time calibration, covering about 6000 mammalian species.<span class="citation" data-cites="Upham2019InfMam"><sup><strong>Upham2019InfMam?</strong></sup></span> Faith’s PD measures the sum of unique branches from an arbitrary root to a set of tips, and comparatively larger values indicate a more phylogenetic diverse species pool. We measured phylogenetic diversity starting from the root of the entire tree (and not from Chiroptera); this bears no consequences on the resulting values, since all branches leading up to Chiroptera are only counted once per species pool, and (as we explain when describing the assembly of the composite risk map), all individual risk components are ranged in [0,1]. This measure incorporates a richness component, which we chose not to correct for; the interpretation of the phylogenetic diversity is therefore a weighted species richness, that accounts for phylogenetic over/under-dispersal in some places.</p>
            <h2 id="bat-compositional-uniqueness">Bat compositional uniqueness</h2>
            <p>For every species pool, we measured its Local Contribution to Beta-Diversity;<span class="citation" data-cites="Legendre2013BetDiv"><sup><strong>Legendre2013BetDiv?</strong></sup></span> LCBD works from a species-data matrix (traditionally noted as <span class="math inline"><strong>Y</strong></span>), where species are rows and sites are columns, and a value of 1 indicates occurrence. We extracted the Y matrix assuming that every pixel represents a unique location, and following best practices<span class="citation" data-cites="Legendre2019SpaTem"><sup><strong>Legendre2019SpaTem?</strong></sup></span> transformed it using Hellinger’s distance to account for unequal bat richness at different pixels. The correction of raw community data is particularly important for two reasons: first, it prevents the artifact of richer sites having higher importance; second, it removes the effect of overall species richness, which is already incorporated in the phylogenetic diversity component. High values of LCBD indicate that the pixel has a community that is on average more dissimilar in species composition than what is expected knowing the entire matrix, i.e. a more unique community. Recent results by<span class="citation" data-cites="Dansereau2022EvaEco"><sup><strong>Dansereau2022EvaEco?</strong></sup></span> shows that LCBD measures are robust with regards to spatial scale, and are therefore applicable at the global scale.</p>
            <h2 id="viral-sharing-between-hosts">Viral sharing between hosts</h2>
            <p>For all bat hosts of <em>Betacoronavirus</em>, we extracted their predicted viral sharing network, generated from a previously published generalized additive mixed model of virus sharing by a tensor function of phylogenetic distance and geographic range overlap across mammals.<span class="citation" data-cites="Albery2020PreGlo"><sup><strong>Albery2020PreGlo?</strong></sup></span> This network stores pairwise values of viral community similarity, measured for all hosts (to maintain consistency with teh phylogenetic diversity measure) across all viruses; therefore, we consider that it accounts for some overall similarity in the way hosts deal with viruses, and not only betacoronaviruses. There is empirical evidence that capacity for cross-species transmission even between divergent species is generally high,<span class="citation" data-cites="Mollentze2020Viral"><sup>20</sup></span> especially for beta-coronaviruses.<span class="citation" data-cites="Latinne2020Origin"><sup>13</sup></span> To project viral sharing values into a single value for every pixel, we averaged the pairwise scores. High values of the average sharing propensity means that this specific extant bat assemblage is likely to be proficient at exchanging viruses.</p>
            <h2 id="composite-risk-map">Composite risk map</h2>
            <p>To visualize the aggregated risk at the global scale, we combine the three individual risk components (phylogenetic diversity, compositional uniqueness, and viral sharing) using an additive color model.<span class="citation" data-cites="Seekell2018GeoLak"><sup><strong>Seekell2018GeoLak?</strong></sup></span> In this approach, every risk component gets assigned a component in the RGB color model (phylogenetic diversity is green, compositional uniqueness is red, and viral sharing is blue). In order to achieve a valid RGB measure, all components are re-scaled to the [0,1] interval, so that a pixel with no sharing, no phylogenetic diversity, and no compositional uniqueness is black, and a pixel with maximal values for each is white. This additive model conveys both the intensity of the overall risk, but also the nature of the risk as colors diverge towards combinations of values for three risk components. Out of the possible combinations, the most risky in terms or rapid diversification and spillover potential is high phylogenetic diversity and low viral sharing,<span class="citation" data-cites="Gomulkiewicz2000HotSpo"><sup><strong>Gomulkiewicz2000HotSpo?</strong></sup></span> in that this allows multiple independent host-virus coevolutionary dynamics to take place in the same location. In the colorimetric space, this correspond to yellow – because the HSV space is more amenable to calculations for feature extraction,<span class="citation" data-cites="Keke2010StuSki"><sup><strong>Keke2010StuSki?</strong></sup></span> we measured the risk level by calculating the angular distance of the hue of each pixel to a reference value of 60 (yellow), and weighted this risk level by the value component. Specifically, given a pixel with colorimetric coordinates <span class="math inline">(<em>h</em>, <em>s</em>, <em>v</em>)</span>, its ranged weighted risk value is</p>
            <p><span class="math display">$$
            v\times\left[1-\frac{\left|\text{atan}\left(\text{cos}(\text{rad}(h)), \text{sin}(\text{rad}(h))\right) - X\right|}{2\pi}\right]\,,
            $$</span></p>
            <p>where X is <span class="math inline">atan(cos(rad(60)),sin(rad(60)))</span>, a constant approximately equal to <span class="math inline">0.5235</span>.</p>
            <h2 id="viral-phylogeography-and-evolutionary-diversification">Viral phylogeography and evolutionary diversification</h2>
            <p>To next represent phylogeography of betacoronaviruses in bats, we aggregated and analyzed betacoronavirus sequence data. We used the following query to pull all <em>Betacoronavirus</em> sequence data from the GenBank Nucleotide database except SARS-CoV-2; (“Betacoronavirus”[Organism] OR betacoronavirus[All Fields]) NOT (“Severe acute respiratory syndrome coronavirus 2”[Organism] OR sars-cov-2[All Fields]). We added a single representative sequence for SARS-CoV-2 and manually curated to remove sequences without the RNA-dependent RNA polymerase (RdRp) sequence or that contained words indicating recombinant or laboratory strains including “patent”, “mutant”, “GFP”, and “recombinant”. We filtered over-represented taxa including betacoronavirus 1, hCoV-OC43, Middle East respiratory syndrome coronavirus, Murine hepatitis virus, and hCoV-HKU1. Curated betacoronavirus RdRp sequences were then aligned using MAFFT<span class="citation" data-cites="Katoh2013MafMul"><sup><strong>Katoh2013MafMul?</strong></sup></span> v1.4.0 (Algorithm FFT-NS-2, Scoring matrix 200PAM / k=2, gap open penalty 1.53m offset value 0.123) and a maximum likelihood tree reconstructed in IQ-TREE<span class="citation" data-cites="Nguyen2015IqtFas"><sup><strong>Nguyen2015IqtFas?</strong></sup></span> v1.6.12 with ModelFinder<span class="citation" data-cites="Kalyaanamoorthy2017ModFas"><sup><strong>Kalyaanamoorthy2017ModFas?</strong></sup></span> ultrafast bootstrap approximation<span class="citation" data-cites="Hoang2018UfbImp"><sup><strong>Hoang2018UfbImp?</strong></sup></span> with a general time reversible model with empirical base frequencies and the 5-discrete-rate-category FreeRaye model of nucleotide substitution (GTR+F+R5).</p>
            <p>We first tested the hypothesis that hotspots of viral diversification would track hotspots of bat diversification. To do so, we plotted the number of known bat hosts (specifically only those included in the phylogeny, so there was a 1:1 correspondence between data sources) against the “mean evolutionary distinctiveness” of the associated viruses. To calculate this, we derived the fair proportions evolutionary distinctiveness<span class="citation" data-cites="Isaac2007MamEdg"><sup><strong>Isaac2007MamEdg?</strong></sup></span> for each of the viruses in the tree, then averaged these at the bat species level, projected these values onto their geographic distributions, and averaged across every bat found in a given pixel. As such, this can be thought of as a map of the mean evolutionary distinctiveness of the known viral community believed to be associated with a particular subset of bats present.</p>
            <h2 id="co-distribution-of-hosts-and-viral-hotspots">Co-distribution of hosts and viral hotspots</h2>
            <p>Subsequently, we tested the hypothesis that the biogeography of bat betacoronaviruses should track the biogeography of their hosts. To test this idea, we loosely adapted a method from,<span class="citation" data-cites="Kreft2007GloPat Kreft2010FraDel"><sup><strong>Kreft2007GloPat?</strong>,<strong>Kreft2010FraDel?</strong></sup></span> who proposed a phylogenetic method for the delineation of animal biogeographic regions. In their original method, a distance matrix - where each row or column represents a geographic raster’s grid cell, and the dissimilarity values are the “beta diversity similarity” of their community assemble - undergoes non-metric multidimensional scaling (NMDS); the first two axes of the NMDS are projected geographically using a four-color bivariate map. Here, we build on this idea with an entirely novel methodology. First, we measure the phylogenetic distance between the different viruses in the betacoronaviruses tree by using the cophenetic function in <code>ape</code>;<span class="citation" data-cites="Paradis2019ApeEnv"><sup><strong>Paradis2019ApeEnv?</strong></sup></span> subsequently, we take a principal components analysis of that distance matrix (readily interchangeable for NMDS in this case) to project the viral tree into an n-dimensional space. We then take the first two principal components and, as with the evolutionary distinctiveness analysis, aggregated these to a mean host value and projected them using a four-color bivariate map.</p>
            <h2 id="data-availability-statement">Data availability statement</h2>
            <p>The code to reproduce these analyses, as well as the data (with the exception of the IUCN rangemaps, which must be downloaded from their website) are available in the <a href="https://github.com/viralemergence/betamap"><code>viralemergence/betamap</code></a> repository on GitHub.</p>
            <h1 class="unnumbered" id="references">References</h1>
            <div id="refs" class="references csl-bib-body" data-line-spacing="2" role="doc-bibliography">
            <div id="ref-Plowright2017Pathways" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">1. </div><div class="csl-right-inline">Plowright, R. K. <em>et al.</em> Pathways to zoonotic spillover. <em>Nature Reviews Microbiology</em> <strong>15</strong>, 502–510 (2017).</div>
            </div>
            <div id="ref-Agosta2010How" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">2. </div><div class="csl-right-inline">Agosta, S. J., Janz, N. &amp; Brooks, D. R. How specialists can be generalists: Resolving the <span>‘parasite paradox’</span> and implications for emerging infectious disease. <em>Zoologia (Curitiba)</em> <strong>27</strong>, 151–162 (2010).</div>
            </div>
            <div id="ref-Power2004Pathogen" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">3. </div><div class="csl-right-inline">Power, A. G. &amp; Mitchell, C. E. Pathogen Spillover in Disease Epidemics. <em>The American Naturalist</em> <strong>164</strong>, S79–S89 (2004).</div>
            </div>
            <div id="ref-Geoghegan2017Comparative" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">4. </div><div class="csl-right-inline">Geoghegan, J. L., Duchêne, S. &amp; Holmes, E. C. Comparative analysis estimates the relative frequencies of co-divergence and cross-species transmission within viral families. <em>PLOS Pathogens</em> <strong>13</strong>, e1006215 (2017).</div>
            </div>
            <div id="ref-Thompson2005Geographic" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">5. </div><div class="csl-right-inline">Thompson, J. N. <em>The Geographic Mosaic of Coevolution</em>. (University Of Chicago Press, 2005).</div>
            </div>
            <div id="ref-Thompson1994Coevolutionary" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">6. </div><div class="csl-right-inline">Thompson, J. N. <em>The Coevolutionary Process</em>. (University of Chicago Press, 1994).</div>
            </div>
            <div id="ref-Janzen1980When" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">7. </div><div class="csl-right-inline">Janzen, D. H. When is it Coevolution? <em>Evolution</em> <strong>34</strong>, 611–612 (1980).</div>
            </div>
            <div id="ref-Price2002Macroevolutionary" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">8. </div><div class="csl-right-inline">Price, P. W. <em>Macroevolutionary Theory on Macroecological Patterns</em>. (Cambridge University Press, 2002). doi:<a href="https://doi.org/10.1017/CBO9780511615030">10.1017/CBO9780511615030</a>.</div>
            </div>
            <div id="ref-Gomulkiewicz2007Dos" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">9. </div><div class="csl-right-inline">Gomulkiewicz, R. <em>et al.</em> Dos and don’ts of testing the geographic mosaic theory of coevolution. <em>Heredity</em> <strong>98</strong>, 249–258 (2007).</div>
            </div>
            <div id="ref-Thrall2007Coevolution" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">10. </div><div class="csl-right-inline">Thrall, P. H., Hochberg, M. E., Burdon, J. J. &amp; Bever, J. D. Coevolution of symbiotic mutualists and parasites in a community context. <em>Trends in Ecology &amp; Evolution</em> <strong>22</strong>, 120–126 (2007).</div>
            </div>
            <div id="ref-Anthony2017Global" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">11. </div><div class="csl-right-inline">Anthony, S. J. <em>et al.</em> Global patterns in coronavirus diversity. <em>Virus Evolution</em> <strong>3</strong>, (2017).</div>
            </div>
            <div id="ref-Ruiz-Aravena2022Ecology" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">12. </div><div class="csl-right-inline">Ruiz-Aravena, M. <em>et al.</em> Ecology, evolution and spillover of coronaviruses from bats. <em>Nature Reviews Microbiology</em> <strong>20</strong>, 299–314 (2022).</div>
            </div>
            <div id="ref-Latinne2020Origin" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">13. </div><div class="csl-right-inline">Latinne, A. <em>et al.</em> Origin and cross-species transmission of bat coronaviruses in China. <em>bioRxiv: The Preprint Server for Biology</em> 2020.05.31.116061 (2020) doi:<a href="https://doi.org/10.1101/2020.05.31.116061">10.1101/2020.05.31.116061</a>.</div>
            </div>
            <div id="ref-Anthony2013Coronaviruses" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">14. </div><div class="csl-right-inline">Anthony, S. J. <em>et al.</em> Coronaviruses in bats from Mexico. <em>The Journal of General Virology</em> <strong>94</strong>, 1028–1038 (2013).</div>
            </div>
            <div id="ref-Goes2013Novel" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">15. </div><div class="csl-right-inline">Góes, L. G. B. <em>et al.</em> Novel Bat Coronaviruses, Brazil and Mexico. <em>Emerging Infectious Diseases</em> <strong>19</strong>, 1711–1713 (2013).</div>
            </div>
            <div id="ref-Goes2016Genetic" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">16. </div><div class="csl-right-inline">Góes, L. G. B. <em>et al.</em> Genetic diversity of bats coronaviruses in the Atlantic Forest hotspot biome, Brazil. <em>Infection, Genetics and Evolution: Journal of Molecular Epidemiology and Evolutionary Genetics in Infectious Diseases</em> <strong>44</strong>, 510–513 (2016).</div>
            </div>
            <div id="ref-Brandao2008Coronavirus" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">17. </div><div class="csl-right-inline">Brandão, P. E. <em>et al.</em> A coronavirus detected in the vampire bat Desmodus rotundus. <em>Brazilian Journal of Infectious Diseases</em> <strong>12</strong>, 466–468 (2008).</div>
            </div>
            <div id="ref-Muylaert2022Present" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">18. </div><div class="csl-right-inline">Muylaert, R. L. <em>et al.</em> Present and future distribution of bat hosts of sarbecoviruses: Implications for conservation and public health. <em>Proceedings of the Royal Society B: Biological Sciences</em> <strong>289</strong>, 20220397 (2022).</div>
            </div>
            <div id="ref-Ka-WaiHui2006Reasons" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">19. </div><div class="csl-right-inline">Ka-Wai Hui, E. Reasons for the increase in emerging and re-emerging viral infectious diseases. <em>Microbes and Infection</em> <strong>8</strong>, 905–916 (2006).</div>
            </div>
            <div id="ref-Mollentze2020Viral" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">20. </div><div class="csl-right-inline">Mollentze, N. &amp; Streicker, D. G. Viral zoonotic risk is homogenous among taxonomic orders of mammalian and avian reservoir hosts. <em>Proceedings of the National Academy of Sciences</em> <strong>117</strong>, 9423–9430 (2020).</div>
            </div>
            </div>
        </article>
    </main>

    <footer>
        <div class="container">
            <h2>Affiliations</h2> 
            <sup>1</sup>&nbsp;Université de Montréal; <sup>2</sup>&nbsp;Québec Centre for Biodiversity Sciences; <sup>3</sup>&nbsp;Molecular Epidemiology and Public Health Laboratory, School of Veterinary Science, Massey University, New Zealand; <sup>4</sup>&nbsp;Paul G. Allen School for Global Health, Washington State University, Pullman, WA, United States; <sup>5</sup>&nbsp;Department of Biology, Georgetown University, Washington, DC, USA; <sup>6</sup>&nbsp;Department of Biology, University of Oklahoma, Norman, OK, USA; <sup>7</sup>&nbsp;Department of Biology, Georgetown University, Washington, DC,; <sup>8</sup>&nbsp;Center for Global Health Science and Security, Georgetown University Medical Center, Washington, DC, USA; <sup>9</sup>&nbsp;Department of Microbiology and Immunology, Georgetown University Medical Center, Washington, DC, USA
            <br />
            <sup>‡</sup>&nbsp;These authors contributed equally to the work
        </div>
    </footer>
    
</body>
</html>