<!DOCTYPE html>
<html lang="" dir="ltr">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>The coevolutionary mosaic of betacoronavirus emergence risk</title>
    <script src="https://hypothes.is/embed.js" async></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
    integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />
    <style>
        @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,400;0,500;1,400;1,500&display=swap');

        :root {
            --citation: #1976D2;
            --structure: hsl(210, 61%, 43%);
            
            --light: #474747;
            --pale: #a5a5a5;
            --grey: #e0e0e0;
            --nav: #fdfdfd;

            --width: 820px;
            --inner-space: 1.2em;
        }

        body {
            font-family: "Roboto", sans-serif;
            margin: 0 0;
            padding: 0 0;
        }

        article > p {
            line-height: 135%;
        }
        article > h1 {
            padding-top: 3em;
        }
        a {
            text-decoration: none;
            color: #000;
            border-bottom: 1px solid var(--structure);
        }
        a:hover {
            font-weight: 500;
        }
        .citation {
            color: var(--citation);
        }
        .container {
            margin: 0em auto;
            max-width: var(--width);
            padding: var(--inner-space);
        }
        header > .container {
            margin-top: 4em;
        }
        code, pre {
            font-family: "Roboto Mono", monospace;
        }
        pre {
            white-space: pre-wrap;
        }
        h1, h2, h3 {
            color: var(--structure);
            font-family: "Roboto Condensed", sans-serif;
            font-weight: 400;
        }
        header {
            font-family: "Roboto Condensed", sans-serif;
            z-index: 5;
            color: black;
            margin-top: 4rem;
        }
        .header--title {
            font-weight: 400;
            font-size: 2rem;
            margin-bottom: 1rem;
        }
        .header--authors > span {
            color: var(--light);
            font-weight: 400;
        }
        main {
            padding-top: 8em !important;
            padding-bottom: 8em !important;
        }
        footer {
            background-color: var(--grey);
            font-weight: 300;
        }
        .abstract {
            color: var(--light);
            margin-bottom: 5em;
        }
        figure {
            max-width: 100%;
            border: 1px solid var(--pale);
            border-radius: 2px;
            margin: 1em 0.5em;
        }
        figure > img {
            max-width: 100%;
            display: block;
            margin: 0em auto;
        }
        figure > figcaption {
            padding: var(--inner-space);
        }
        table {
            width: 100%;
            min-width: 100%;
            border-collapse: collapse;
        }
        table > caption {
            padding: var(--inner-space);
        }
        figcaption, caption {
            color: var(--light);
            font-weight: 300;
        }
        thead {
            color: var(--structure);
        }
        td, th {
            padding: 3px;
        }
        tr {
            border-top: 1px solid var(--structure);
            border-bottom: 1px solid var(--structure);
        }
        tbody > tr:nth-child(even) {background: var(--grey)};
        tbody {
            border-bottom: 1px solid var(--structure);
        }
        #refs {
            font-size: 90%;
            column-count: 2;
        }
        #refs > .csl-entry {
            padding-bottom: 0.2em;
            padding-top: 0.2em;
        }
        .hanging-indent > .csl-entry {
            padding-left: 15px ;
            text-indent: -15px ;
        }    </style>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/katex.min.css"
    integrity="sha384-D+9gmBxUQogRLqvARvNLmA9hS2x//eK1FhVb9PiU86gmcrBrJAQT8okdJ4LMp2uv" crossorigin="anonymous" />
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/katex.min.js"
    integrity="sha384-483A6DwYfKeDa0Q52fJmxFXkcPCFfnXMoXblOkJ4JcA8zATN6Tm78UNL72AKk+0O"
    crossorigin="anonymous"></script>
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/contrib/auto-render.min.js"
    integrity="sha384-yACMu8JWxKzSp/C1YV86pzGiQ/l1YUfE8oPuahJQxzehAjEt2GiQuy/BIvl9KyeF"
    crossorigin="anonymous"></script>
    <script>
        document.addEventListener("DOMContentLoaded", function () {
            renderMathInElement(document.body, {
                delimiters: [
                { left: "$$", right: "$$", display: true },
                { left: "$", right: "$", display: false }
                ]
            });
        });
    </script>
</head>

<body>
    <header>
        <div class="container">
            <div class="header--title">The coevolutionary mosaic of betacoronavirus emergence risk</div>
            <div class="header--authors">
                <span class="header--author">
                Norma Forero Rocio Munoz
                <sup class="affilcode">1</sup>
                <sup class="affilcode">2</sup>
                <sup class="affilcode">‡</sup>
                </span><span class="header--author">
                Renata L. Muylaert
                <sup class="affilcode">3</sup>
                </span><span class="header--author">
                Stephanie N. Seifert
                <sup class="affilcode">4</sup>
                </span><span class="header--author">
                Gregory F. Albery
                <sup class="affilcode">5</sup>
                </span><span class="header--author">
                Daniel J. Becker
                <sup class="affilcode">6</sup>
                </span><span class="header--author">
                Colin J. Carlson
                <sup class="affilcode">7</sup>
                <sup class="affilcode">8</sup>
                <sup class="affilcode">9</sup>
                <sup class="affilcode">‡</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-0735-5184" target="_blank">
                    Timothée Poisot
                </a>
                <sup class="affilcode">1</sup>
                <sup class="affilcode">2</sup>
                <sup class="affilcode">‡</sup>
                </span>
            </div>
        </div>
    </header>
    
    <main class="container">
        <section class="abstract">
            Driven by the need to understand the ecological factors involved in the emergence of <em>Betacoronavirus</em> (the genus causing the SARS, MERS, and COVID-19 in human) through bat hosts, we develop an approach to the assesment of spillover risk based on the Geographic Mosaic Theory of Coevolution. In doing so, we provide a global mapping of the spillover risk posed by betacoronaviruses, reflecting the fact that this risk is best understood through the multi-faceted prism of ecological and evolutionary mechanisms. Our framework reveals that host richness alone, although a component of viral hazard, is not a sufficiently integrative predictor of risk. We offer alternative insights based on viral sharing, host compositional uniqueness, and host phylogenetic diversity and phylogeographic regions. By comparing our aggregated measure of risk to a proxy for human density, namely the proportion of each pixel that is covered by urban or built land, we provide a synthetic risk map, allowing the identification of hotspots where the bat-betacoronavirus interaction network may facilitate the emergence of novel viruses and their spillover into human populations.


            <div class="artifacts">
                <a href="draft.pdf" title="Draft PDF" target="_blank">draft</a>
                <a href="preprint.pdf" title="Preprint PDF" target="_blank">preprint</a>
            </div>
        </section>
        
        <article>
            <p>Disease emergence is complex, and is driven not only by animal-human contact, but also by the underlying evolutionary dynamics in viral reservoirs.<span class="citation" data-cites="Plowright2017PatZoo"><sup>1</sup></span> Although host richness is often used as a superficial proxy for spillover risk,<span class="citation" data-cites="Anthony2017GloPat Ruiz-Aravena2022EcoEvo"><sup>2,3</sup></span> these approaches oversimplify the relevant interspecific heterogeneity in immunology, behavior, and other traits, and therefore overlook unique host pools that allow for the rapid evolution of highly divergent viruses.<span class="citation" data-cites="Agosta2010HowSpe"><sup>4</sup></span> In the case of generalist pathogens like betacoronaviruses, there is conceptual and empirical support to the idea that these community-level mechanisms are even more important,<span class="citation" data-cites="Power2004PatSpi"><sup>5</sup></span> particularly given that cross-species transmission may, as a rule, structure viral evolution more than co-divergence with hosts.<span class="citation" data-cites="Geoghegan2017ComAna"><sup>6</sup></span> This creates a disconnect between coevolutionary theory (including empirical evidence from virology) and most existing ecological frameworks for mapping spillover risk.</p>
            <p>The Geographic Mosaic Theory of Coevolution (GMTC) attempts to explicitly connect microevolutionary dynamics to the macroecology and biogeography of symbiotic interactions.<span class="citation" data-cites="Thompson2005GeoMos"><sup>7</sup></span> The GMTC posits that coevolutionary processes among pairs<span class="citation" data-cites="Thompson1994CoePro"><sup>8</sup></span> or complexes<span class="citation" data-cites="Janzen1980WheIt"><sup>9</sup></span> of species are structured in space by the rippling effects of abiotic conditions onto evolutionary mechanism, giving rise to fragmented systems with different structure and ecologically dynamics over large spatial extents.<span class="citation" data-cites="Price2002MacThe"><sup>10</sup></span> The GMTC predicts a spatial fragmentation of coevolutionary dynamics under the joint action of three processes:<span class="citation" data-cites="Gomulkiewicz2007DosDon"><sup>11</sup></span> coevolutionary hot- and coldspots, which appear when the intensity of <em>interaction</em> (in terms of reciprocal fitness consequences) varies spatially; selection mosaics, wherein the intensity of <em>selection</em> varies across space, driven by both the biotic complexity of the community (locally diverse hosts and viruses are more biotically complex) and the local favorability of the environment;<span class="citation" data-cites="Thrall2007CoeSym"><sup>12</sup></span> and trait remixing, which occurs when coevolutionary dynamics are driven by by the arrival (or departure) of <em>functional traits</em>, through changes in community composition due to invasions, meta-community dynamics, and disperal.</p>
            <p>Here, we apply the GMTC to explore and explain the global biogeography of betacoronaviruses, the group that includes SARS-CoV, MERS-CoV, and SARS-CoV-2. In their bat reservoirs, coronaviruses evolve through a mix of host jumps, recombination among disparate lineages, and, to a lesser degree, co-divergence with their hosts—<span class="citation" data-cites="Anthony2017GloPat"><sup>2</sup></span>a mix of mechanisms that creates a complex and nonlinear relationship between host diversity and viral emergence. Working from a recently published database of bat hosts of betacoronaviruses, we develop the first global maps of both host and virus evolutionary distinctiveness and biogeographic regions for this system. Aiming to explain these patterns, we develop a generalized framework for applying the GMTC to host-virus interactions, with a specific emphasis on the potential to create independent coevolutionary dynamics (and therefore spatial fragmentation in risk) through heterogeneity. We develop a trivariate risk assessment system that connects each GMTC mechanism to a quantifiable aspect of host-virus interactions: (i) viral sharing rates in host communities, representing the strength of potential interaction between viruses and any one host (i.e., places where viruses undergo constant host switching may be coevolutionary coldspots); (ii) the phylogenetic diversity of hosts, as a proxy for variation in the immunological mechanisms that antagonize viruses (i.e., the selection mosaic); and (iii) the local uniqueness of the bat community, representing the potential for viruses to be exposed to novel host traits (e.g., variation in receptor sequences). Together, we argue that these can be used to identify and map the evolutionary drivers that—in conjunction with transmission processes (e.g., viral prevalence in reservoirs and animal-human contact rates)— determine disease emergence risk.</p>
            <h1 id="results-and-discussion">Results and Discussion</h1>
            <h2 id="hotspots-of-host-richness-and-viral-diversification-are-distinct">Hotspots of host richness and viral diversification are distinct</h2>
            <p>Bats, the second most diverse groups of mammals, are found worldwide; gradients in their species richness generally track broader patterns of mammal diversity, with a striking Neotropical hotspot (especially in the Amazon basin) and a secondary hotspot centered in the southeast Asian peninsula. These hotspots of bat diversity are generally presumed to be hotspots of viral adaptive radiation, and therefore areas of concern for human health.<span class="citation" data-cites="Anthony2017GloPat Olival2017HosVir"><sup>2,13</sup></span> However, the hotspots of bat betacoronavirus reservoirs show a distinct pattern, with primary hotspots (both in terms of size and higher values) of host richness situated primarily southeast Asia, parts of southern Europe, and to a lesser extent parts of Africa in the -25-0 range of latitudes (fig. 1; top). Although hundreds of species likely host undiscovered betacoronaviruses, machine learning predictions have suggested that these undiscovered reservoirs should follow the same diversity gradient.<span class="citation" data-cites="Becker2022OptPre"><sup>14</sup></span> In principle, these hotspots of locally-diverse, virus-rich bat communities should drive more adaptive diversification in their viruses.</p>
            <figure>
            <img src="figures/combined_richness.png" id="fig:richness" alt="Figure 1: Top panel: relative diversity of known bat hosts of betacoronaviruses. This map shows that the region with the largest number of possible hosts is South-Eastern Asia. Bottom panel: congruence between the evolutionary distinctiveness of the hosts (grey to blue) and the viruses (grey to red)." /><figcaption aria-hidden="true">Figure 1: Top panel: relative diversity of known bat hosts of betacoronaviruses. This map shows that the region with the largest number of possible hosts is South-Eastern Asia. Bottom panel: congruence between the evolutionary distinctiveness of the hosts (grey to blue) and the viruses (grey to red).</figcaption>
            </figure>
            <p>However, we find that the global pattern of betacoronavirus phylogenetic distinctiveness is quite distinct from both bat host richness and phylogenetic distinctiveness (fig. 1; bottom). In contrast to the sparsity of Neotropical betacoronavirus hosts, South America has the most evolutionary distinct hosts <em>and</em> viruses, followed by secondary hotspots in southeast Asia and the Rift Valley region have mostly distinct viruses. Some degree of sampling bias may contribute to these patterns: for example, South-America is one of the places where the fewest bat betacoronavirus sequences have been generated,<span class="citation" data-cites="Anthony2017GloPat Allen2017GloHot Olival2017HosVir"><sup>2,13,15</sup></span> resulting in a sparser phylogenetic tree, and artificially inflating distinctiveness; conversely, disproportionate research effort in eastern China<span class="citation" data-cites="Cohen2022SamStr"><sup>16</sup></span> may have led to a more complete inventory of the local diversity of coronaviruses, again inflating these metrics relative to underlying patterns. Even accounting for these potential biases, though, there is obvious heterogeneity in betacoronavirus evolutionary distinctiveness that is distinct from overall bat diversity.</p>
            <p>On closer inspection, these patterns recapitulate the evolutionary history of both the order Chiroptera and the genus <em>Betacoronavirus</em>. Horseshoe bats (Rhinolophidae) are both the reservoirs of the SARS-like viruses (subgenus <em>Sarbecovirus</em>) and the presumed ancestral host of <em>Betacoronavirus</em>.<span class="citation" data-cites="Latinne2020OriCro"><sup>17</sup></span> The hotspot of host richness and viral diversity in southeast Asia—both of which are disproportionately high, considering the global landscape of bat species richness—is almost entirely driven by adaptive radiation in this clade<span class="citation" data-cites="Becker2022OptPre Ruiz-Aravena2022EcoEvo"><sup>3,14</sup></span>. In contrast, the Neotropical hotspot of viral distinctiveness is the result of isolation by vicariance for both bats and their viruses. Out of the four main groups of betacoronaviruses, only the subgenus <em>Merbecovirus</em> has been found in animals in the Americas—an introduction that is generally presumed to be ancient. While comparatively understudied, New World merbecoviruses have been found in the ghost-faced bats (Mormoopidae), New World leaf-nosed bats (Phyllostomidae), and free-tailed bats (Molossidae) (add cite: Olival 2020 PLoS Pathogens). The former two groups are endemic to the Neotropics, while the latter two and particularly the phyllostomids are responsible for the hotspot of bat diversity in the Amazon. Together, these groups play host to a distinct regime of bat-betacoronavirus coevolution in the Americas.</p>
            <h2 id="the-phylogeographic-regions-of-hosts-and-their-viruses-overlap">The phylogeographic regions of hosts and their viruses overlap</h2>
            <p>Despite differences in hotspots of evolutionary distinctiveness, there are reasons to expect that the phylogeography of bats and betacoronaviruses should show some degree of congruence.<span class="citation" data-cites="VanBrussel2022ZooDis Letko2020BatVir"><sup>18,19</sup></span> In particular, this should be the case if viruses can circulate among hosts and co-evolve with local hosts communities, making their evolutionary process more than a byproduct of host evolution. High density of hosts sharing the same virus (albeit possibly different strains) can drive or result from evolution of the bat antiviral immune system, resulting in spatially distinct immunological responses, as evidenced in several bat species.<span class="citation" data-cites="Banerjee2020NovIns"><sup>20</sup></span> Immune characteristics that allow bats to be better adapted to infection by emerging viruses,<span class="citation" data-cites="Gorbunova2020WorGoe Irving2021LesHos"><sup>21,22</sup></span> in addition to being hardcoded in their genome,<span class="citation" data-cites="Jebb2020SixRef"><sup>23</sup></span> may be related to a wide variety of diets,<span class="citation" data-cites="Banerjee2020NovIns MorenoSantillan2021LarGen"><sup>20,24</sup></span> themselves likely to be driven by spatial effects, especially at the local scale – bats, indeed, occupy a variety of environments, and therefore display a variety of adaptations to these environments.<span class="citation" data-cites="Muylaert2022PreFut"><sup>25</sup></span></p>
            <figure>
            <img src="figures/combined_biogeo.png" id="fig:biogeo" alt="Figure 2: Phylogeographic regions of bats (top) and viruses (bottom) based on the joint analysis of their occurrence and phylogenetic relatedness. The different colors show tendencies to separate alongside the first two components of a PCoA. Note that the PCoA for the bats and viruses are independent, and so cannot be compared directly – that being said, the regions can be compared across maps." /><figcaption aria-hidden="true">Figure 2: Phylogeographic regions of bats (top) and viruses (bottom) based on the joint analysis of their occurrence and phylogenetic relatedness. The different colors show tendencies to separate alongside the first two components of a PCoA. Note that the PCoA for the bats and viruses are independent, and so cannot be compared directly – that being said, the regions can be compared across maps.</figcaption>
            </figure>
            <p>In fig. 2, we show a projection of the phylogeographic signal of bats (top) and viruses (bottom) in space; the distinct groupings (represented by different colors symbolizing positions in the subspace formed by the first two axes of the PCoA) are essentially equivalent between the two groups, and can be coarsely delineated as southeast Asia, Eurasia above a northing of 25, and Africa and south America. These results suggest that, although the evolutionary distinctiveness of the bat-betacoronaviruses complex varies spatially, the system shows an important degree of spatial consistency, with a reduced number of bioregions. Available information describing the spillover of zoonotic betacoronaviruses of bat origin where data was available before and up through the COVID-19 pandemic puts spillover events of SARS-CoV-2 in Wuhan, China; SARS-CoV in Guangdong, China based on the presence of closest known viruses circulating in nature, and a nearby location where serological (antibody) evidence has indicated human exposure to SARS-like viruses;<span class="citation" data-cites="Wang2018SerEvi"><sup>26</sup></span> MERS-CoV in Saudi Arabia based on index cases available from a recently-published compendium of cases.<span class="citation" data-cites="Ramshaw2019DatGeo"><sup>27</sup></span> For the latest event, most if not all index cases are presumed to be camel-to-human transmission, and the precise origin point (if it exists) of MERS-CoV in bats is uncertain. Recent recombinant canine coronavirus spillover events in Haiti<span class="citation" data-cites="Lednicky2021IsoNov"><sup>28</sup></span> and Europe<span class="citation" data-cites="Vlasova2022AniAlp"><sup>29</sup></span> are not relevant here, as bats’ involvement in these cycles of transmission have been supposed to be non-existent. These index cases fall within different phylogeographic bioregions (fig. 2), which further highlight the issue that different host-virus sub-systems may lead to widespread emergence.</p>
            <h2 id="coevolution-informed-emergence-risk-is-different-in-space">Coevolution-informed emergence risk is different in space</h2>
            <p>As host richness, joint distinctiveness, or phylogeographic structure suggest that the bat-betacoronaviruses complex is globally fragmented enough to give rise to both different levels of risk (as evidenced by the spatial location of spillover events) and different types of co-evolutionary dynamics, we turn to the Geographic Mosaic Theory of Coevolution to provide a measure of risk accounting for multiple processes. In fig. 3, we overlapped three components of spillover risk: viral sharing, <em>i.e.</em> the chance that two bats will share viruses overall; Local Contribution to Beta Diversity, <em>i.e.</em> the fact that a bat community is compositionally unique compared to the average compositional similarity across the entire system; finally, host phylogenetic diversity, <em>i.e.</em> how dispersed the bats in a location are within the tree of life. This approach leads to the definition of broad biogeographic regions of risk, where the same color represents the same type of risk. By way of constrat to figures fig. 1 and fig. 2, these regions do not necessarilly overlap with previous spatial partitions of the bat-betacoronaviruses complex.</p>
            <figure>
            <img src="figures/risk_trivariate.png" id="fig:trivariate" alt="Figure 3: Trivariate additive mapping of the components of risk in the red/green/blue, where high virus sharing is encoded in the blue channel, host phylogenetic diversity in the green channel, and compositional uniqueness in the red channel. A pixel that would maximize all measures (highest possible risk) would be a pure white (specifically RGB(1.0, 1.0. 1.0)), and a pixel with the lowest possible values would be pure black (specifically RGB(0.0, 0.0, 0.0)). Therefore, lighter values (the sum of the three channels gets closer to 3) indicate higher risk, and the color indicates the proportional distribution of the factors making up the total risk." /><figcaption aria-hidden="true">Figure 3: Trivariate additive mapping of the components of risk in the red/green/blue, where high virus sharing is encoded in the blue channel, host phylogenetic diversity in the green channel, and compositional uniqueness in the red channel. A pixel that would maximize all measures (highest possible risk) would be a pure white (specifically RGB(1.0, 1.0. 1.0)), and a pixel with the lowest possible values would be pure black (specifically RGB(0.0, 0.0, 0.0)). Therefore, lighter values (the sum of the three channels gets closer to 3) indicate higher risk, and the color indicates the proportional distribution of the factors making up the total risk.</figcaption>
            </figure>
            <p>From the perspective of spillover risk, the most important combination of factors is a high phylogenetic diversity of hosts with low viral sharing; this, essentially, means that very different betacoronaviruses could co-exist within the same place. This is particularly the case given that betacoronaviruses often evolve and even achieve host shifts through recombination, which requires the co-occurrence of sufficiently distinct viruses to be a major driver of emergence. In fig. 3, this corresponds to yellow to pale green areas, which are essentially limited to South-Eastern Asia, and to some part of Sub-Saharan Africa. Adopting a geographic mosaic theory perspective on risk, other regions of the world are of lesser concern (fig. 4). Our risk decomposition does not account for viral diversity or distinctiveness. The simple rationale behind it is that the acquisition of viral data is rarely disconnected from the acquisition of host data. There are more sources of information on hosts than on viruses, allowing to develop a host-centric perspective on risk (although this estimate would more accurate with viral traits related to <em>e.g.</em> ability to switch hosts or pathogenic potential). Areas with high bat diversity and high turnover <em>may</em> facilitate the evolutionary radiation of viruses, matching previous findings that the diversification of bat coronaviruses is driven largely by host shifts (inter-genus or higher levels of cross-species transmission) and, to a lesser degree, cospeciation and sharing, representing intra-genus cross-species transmission.<span class="citation" data-cites="Anthony2017GloPat"><sup>2</sup></span> This diversification is not an actual risk factor for spillover itself, but acts downstream of a spillover event by increasing the random chance of the emergence of a virus with the raw genomic components required for the potential to infect humans.</p>
            <figure>
            <img src="figures/risk_map.png" id="fig:risk" alt="Figure 4: Extraction of a measure of Betacoronavirus spillover risk from bat hosts based on the colorimetric space from fig. 3. The risk is a composite measure of the color value and angular distance to the yellow hue, as defined in the methods, ranged in the unit space. Based on this analyses, regions at high risk of spillover are southeast Asia and Madagascar." /><figcaption aria-hidden="true">Figure 4: Extraction of a measure of <em>Betacoronavirus</em> spillover risk from bat hosts based on the colorimetric space from fig. 3. The risk is a composite measure of the color value and angular distance to the yellow hue, as defined in the methods, ranged in the unit space. Based on this analyses, regions at high risk of spillover are southeast Asia and Madagascar.</figcaption>
            </figure>
            <p>From another perspective, areas of high host uniqueness and virus sharing (red-to-pink) could provide hotspots of <em>Betacoronavirus</em> risk through mixing of unique viruses (via codivergence) and in turn recombination. Under our framework, such a hotspot was identified in Madagascar, where most bat species are endemic following evolutionary divergence from sister species in both African and Asian continents.<span class="citation" data-cites="Shi2014DeeDiv"><sup>30</sup></span> Recent surveillance<span class="citation" data-cites="Kettenburg2022FulGen"><sup>31</sup></span> has identified a novel <em>Betacoronavirus</em> (in the subgenus <em>Nobecovirus</em>) in Madagascar-endemic pteropid bat species (<em>Pteropus rufus</em>, <em>Rousettus madagascariensis</em>), emphasizing strong proof of principle in model predictions.</p>
            <h2 id="human-occupancy-drives-different-levels-of-effective-risk-globally">Human occupancy drives different levels of effective risk globally</h2>
            <p>Based on the previous result, we extracted the risk component from the composite map (see Methods), to provide a single measure of risk varying between 0 and 1. This measure is presented in fig. 4. As this map represents the potential risk, it must be weighed by the potential for contacts with humans. As a proxy for this measure, we used the proportion of build/urban land from the EarthEnv dataset: this is a reasonable proxy for the density of humans per unit area, which increases the probability of pathogen spread more widely.<span class="citation" data-cites="Hazarie2021IntPop"><sup>32</sup></span> Since human activity is required to amplify the frequency of virus encounters and thus create areas of viral amplification, mapping the potential risk against measures of land use is required to generate a more actionable assessment of risk. This map is presented in fig. 5. Most of South America and Europe are at comparatively lower risk, as although densely populated, settlements tend to be in areas with lower potential risk. Regions like Malaysia and the North coast of Australia have a high risk component, but should represent a relatively lower effective risk due to low human density. However, this mapping reveals that South-East Asia, the Indian subcontinent, and parts of sub-Saharan Africa, are at high risk due to the overlap between built areas and bat communities representing more opportunities for cross-species transmission of betacoronaviruses. In looking for the origins of SARS in China,<span class="citation" data-cites="Xu2004EpiClu"><sup>33</sup></span> present serological evidence that strongest human-animal contact results in higher risk of virus exposure, regardless of the animal species, but that different types of contact had different impacts. Ideally, finer-grained information about human activity (rather than human presence through anthropisation) could allow to partition this risk further, alebit at the cost of more hypotheses required to estimate the amount of risk represented by each activity. Our map of purported high risk/diversitifcation potential (Madagascar, South-America) overlay with sampling gaps for <em>Betacoronavirus</em>,<span class="citation" data-cites="Cohen2022SamStr"><sup>16</sup></span> stressing the need for spatially targeted monitoring and discovery.</p>
            <figure>
            <img src="figures/risk_compounded.png" id="fig:compound" alt="Figure 5: Overlap of the percent of each pixel occupied by urbanized structures, representing the degree of settlement, on the spillover risk map (where the risk comes only from wildlife, and ignores multi-hosts chains of transmissions including non-bats hosts). Darker pixels correspond to more risk, in that the GMTC-derived risk of fig. 4 is high and the pixel is densely occupied by human populations. This approach increases the relative risk of several regions in Africa, and highlights the risk in India, southeast China, and the Arabian peninsula where areas of high to moderate risk overlap with areas of denser population." /><figcaption aria-hidden="true">Figure 5: Overlap of the percent of each pixel occupied by urbanized structures, representing the degree of settlement, on the spillover risk map (where the risk comes only from wildlife, and ignores multi-hosts chains of transmissions including non-bats hosts). Darker pixels correspond to more risk, in that the GMTC-derived risk of fig. 4 is high <em>and</em> the pixel is densely occupied by human populations. This approach increases the relative risk of several regions in Africa, and highlights the risk in India, southeast China, and the Arabian peninsula where areas of high to moderate risk overlap with areas of denser population.</figcaption>
            </figure>
            <h1 id="conclusion">Conclusion</h1>
            <p>Bats are important reservoir hosts for different classes of microorganisms, many of which a threat to human health.<span class="citation" data-cites="Letko2020BatVir VanBrussel2022ZooDis"><sup>18,19</sup></span> Chiropterans emerged around 64 million years ago and are one of the most diverse mammalian orders, with an estimated richness of more than 1400 species.<span class="citation" data-cites="Peixoto2018SynEco Simmons2020BatSpe"><sup>34,35</sup></span> They exhibit a broad variety of habitat use, behaviour, and feeding strategies, putting them at key positions in the delivery and provisioning of several ecosystem services, tied to important ecosystem-derived benefits to human.<span class="citation" data-cites="Kasso2013EcoEco"><sup>36</sup></span> For example, bats are an essential component of many seed-dispersal networks.<span class="citation" data-cites="Mello2011MisPar"><sup>37</sup></span> Over two-thirds of bats are know to be either obligate or facultative insectivores, therefore actively contributing for agricultural pest control,<span class="citation" data-cites="Voigt2016BatAnt Williams-Guillen2008BatLim"><sup>38,39</sup></span> and vectors of pathogens that put a risk on human health.<span class="citation" data-cites="Gonsalves2013MosCon Gonsalves2013MosInf"><sup>40,41</sup></span> Because bats are globally distributed and have a long evolutionary history, phylogeographic and biogeographic approaches are required to shed light on the contemporary distribution of coevolutionary processes between bats and the pathogens they host. Not all areas in which bats, viruses, and human are co-occuring are facing a risk of spillover towards human populations, and the areas in which this risk exist may not be facing risks of the same nature and magnitude.</p>
            <p>Here, we propose a simple freamework with broad explanatory power that helps contextualize discoveries like highly divergent nobecoviruses in Madagascar and the previously-neglected adaptive radiation of sarbecoviruses outside of southern China and throughout southeast Asia. In doing so, it advances ecological theory beyond the current state of the art for global maps of emergence risk. For example, previous studies that have used host richness as proxy have predicted a high diversity of unsampled bat viruses,<span class="citation" data-cites="Olival2017HosVir"><sup>13</sup></span> bat coronaviruses,<span class="citation" data-cites="Anthony2017GloPat"><sup>2</sup></span> and even specifically betacoronaviruses<span class="citation" data-cites="Becker2022OptPre"><sup>14</sup></span> in both the Amazon and southeast Asia. While we find that both regions are characterized by highly divergent host and viral communities, our framework identifies key differences between the regions. We find that Latin America is a hotspot of both host and viral distinctiveness, suggesting that this branch of the bat-betacoronavirus complex may be undergoing independent evolutionary dynamics from the rest of the global pool, but with limited potential for viral diversification— a finding that is supported by previous work indicating a higher rate of codivergence in Latin America.<span class="citation" data-cites="Anthony2017GloPat"><sup>2</sup></span> In contrast, in southeast Asia, host richness and viral distinctiveness are high but sharing is low; this suggests a different type of evolutionary dynamics that could generate high local diversity of viruses through host switching and viral recombination (see <em>e.g.</em>,<span class="citation" data-cites="Latinne2020OriCro"><sup>17</sup></span> as well as the discovery of recombinant viruses that share genetic material from both the SARS-CoV and SARS-CoV-2 branches of the Sarbecovirus lineage).<span class="citation" data-cites="Wu2021ComSur"><sup>42</sup></span> Both of these regions are priority areas for sampling, especially given predictions that they contain many bat hosts of undiscovered betacoronaviruses.<span class="citation" data-cites="Becker2022OptPre Cohen2022SamStr"><sup>14,16</sup></span> However, both the evolutionary and ecological aspects of emergence risk are likely higher in southeast Asia—a fact that will only become more relevant, as bats track shifting climates and exchange viruses with other species, creating a hotspot of cross-species transmission unique to the region.<span class="citation" data-cites="Carlson2022CliCha"><sup>43</sup></span></p>
            <p>The diversity and diversification potential of bats responds to anthropogenic factors others than shifting climates.<span class="citation" data-cites="Alves2018GeoVar"><sup>44</sup></span> Land use changes could significantly decrease bat suitability, notably through effects on diet and availability of habitats.<span class="citation" data-cites="Treitler2016EffLoc"><sup>45</sup></span> As our results establish that the diversification of bats betacoronaviruses happens on top of processes affecting hosts, biogeographic variation in human population density and anthropogenic disturbances may feed into co-evolutionary dynamics. Increase in humans-hosts contacts also increase the risk of emergence of novel diseases,<span class="citation" data-cites="Johnson2020GloShi"><sup>46</sup></span> so does the changes in landscape connectivity at local/regional scales.<span class="citation" data-cites="Gryseels2017WheVir"><sup>47</sup></span> This represents a challenge for both conservation strategies and disease ecology: some areas can a high emergence risk and more potential for the acquisition of zoonotic viruses through bat-human encounters.<span class="citation" data-cites="Amman2011InvRol"><sup>48</sup></span> In particular, the challenge ahead lies in the need to quantify actual exposure (and risk) accounting for several transmission scenarios, including both direct and indirect bat - human interactions, and feeding back into the provision of ecosystem services by bats.</p>
            <p><strong>Acknowledgements</strong>: We acknowledge that this study was conducted on land within the traditional unceded territory of the Saint Lawrence Iroquoian, Anishinabewaki, Mohawk, Huron-Wendat, and Omàmiwininiwak nations. This work was supported by funding to the Viral Emergence Research Initiative (VERENA) consortium including NSF BII 2021909 and a grant from Institut de Valorisation des Données (IVADO). This research was enabled in part by support provided by Calcul Québec (www.calculquebec.ca) and Compute Canada (www.computecanada.ca). NF is funded by the NSERC BIOS² CREATE program. TP and NF are funded by the Courtois Foundation. RLM was supported by Bryce Carmine and Anne Carmine (née Percival), through the Massey University Foundation. DJB was supported by the National Institute of General Medical Sciences of the National Institutes of Health (P20GM134973).</p>
            <h1 id="methods">Methods</h1>
            <h2 id="known-betacoronavirus-hosts">Known <em>Betacoronavirus</em> hosts</h2>
            <p>We downloaded the data on bats hosts of <em>Betacoronavirus</em> from <code>https://www.viralemergence.org/betacov</code> on Apr. 2022,<span class="citation" data-cites="Becker2022OptPre"><sup>14</sup></span> and filtered it to “known” hosts (established before the emergence of SARS-CoV-2) and “novel” hosts (confirmed through sampling and competence assays since the initial data collection). The original database was assembled by a combination of data mining and literature surveys, including automated alerts on the “bats” and “coronavirus” keywords to identify novel empirical evidence of bats-betacoronaviruses associations; this yielded a total of 126 known hosts, 47 of which were novel hosts.</p>
            <h2 id="bat-occurrences">Bat occurrences</h2>
            <p>We downloaded the rangemap of every current bat species that was classified as an empirically documented host of <em>Betacoronavirus</em> from the previous step, according to recent IUCN data.<span class="citation" data-cites="IUCN2021IucRed"><sup>49</sup></span> The range maps were subsequently rasterized using the <code>rasterize</code> function from <code>GDAL</code><span class="citation" data-cites="RouaultEven2022GdaOgr"><sup>50</sup></span> at a resolution of approximately 100kmx100km. For every pixel in the resulting raster where at least one bat host of <em>Betacoronavirus</em> was present, we extract the species pool (list of all known bat hosts), which was used to calculate the following risk assessment components: bat phylogenetic diversity, bat compositional uniqueness, and predicted viral sharing risk.</p>
            <h2 id="bat-phylogenetic-diversity">Bat phylogenetic diversity</h2>
            <p>For every pixel, we measured Faith’s Phylogenetic Diversity<span class="citation" data-cites="Faith1992ConEva"><sup>51</sup></span> based on a recent synthetic tree with robust time calibration, covering about 6000 mammalian species.<span class="citation" data-cites="Upham2019InfMam"><sup>52</sup></span> Faith’s PD measures the sum of unique branches from an arbitrary root to a set of tips, and comparatively larger values indicate a more phylogenetic diverse species pool. We measured phylogenetic diversity starting from the root of the entire tree (and not from Chiroptera); this bears no consequences on the resulting values, since all branches leading up to Chiroptera are only counted one per species pool, and (as we explain when describing the assembly of the composite risk map), all individual risk components are ranged in [0,1]. This measure incorporates a richness component, which we chose not to correct for; the interpretation of the phylogenetic diversity is therefore a weighted species richness, that accounts for phylogenetic over/under-dispersal in some places.</p>
            <h2 id="bat-compositional-uniqueness">Bat compositional uniqueness</h2>
            <p>For every species pool, we measured its Local Contribution to Beta-Diversity;<span class="citation" data-cites="Legendre2013BetDiv"><sup>53</sup></span> LCBD works from a species-data matrix (traditionally noted as <span class="math inline"><strong>Y</strong></span>), where species are rows and sites are columns, and a value of 1 indicates occurrence. We extracted the Y matrix assuming that every pixel represents a unique location, and following best practices<span class="citation" data-cites="Legendre2019SpaTem"><sup>54</sup></span> transformed it using Hellinger’s distance to account for unequal bat richness at different pixels. The correction of raw community data is particularly important for two reasons: first, it prevents the artifact of richer sites having higher importance; second, it removes the effect of overall species richness, which is already incorporated in the phylogenetic diversity component. High values of LCBD indicate that the pixel has a community that is on average more dissimilar in species composition than what is expected knowing the entire matrix, i.e. a more unique community. Recent results by<span class="citation" data-cites="Dansereau2022EvaEco"><sup>55</sup></span> shows that LCBD measures are robust with regards to spatial scale, and are therefore applicable at the global scale.</p>
            <h2 id="viral-sharing-between-hosts">Viral sharing between hosts</h2>
            <p>For all bat hosts of <em>Betacoronavirus</em>, we extracted their predicted viral sharing network, generated from a previously published generalized additive mixed model of virus sharing by a tensor function of phylogenetic distance and geographic range overlap across mammals.<span class="citation" data-cites="Albery2020PreGlo"><sup>56</sup></span> This network stores pairwise values of viral community similarity. To project viral sharing values into a single value for every pixel, we averaged the pairwise scores. High values of the average sharing propensity means that this specific extant bat assemblage is likely to be proficient at exchanging viruses.</p>
            <h2 id="composite-risk-map">Composite risk map</h2>
            <p>To visualize the aggregated risk at the global scale, we combine the three individual risk components (phylogenetic diversity, compositional uniqueness, and viral sharing) using an additive color model.<span class="citation" data-cites="Seekell2018GeoLak"><sup>57</sup></span> In this approach, every risk component gets assigned a component in the RGB color model (phylogenetic diversity is green, compositional uniqueness is red, and viral sharing is blue). In order to achieve a valid RGB measure, all components are re-scaled to the [0,1] interval, so that a pixel with no sharing, no phylogenetic diversity, and no compositional uniqueness is black, and a pixel with maximal values for each is white. This additive model conveys both the intensity of the overall risk, but also the nature of the risk as colors diverge towards combinations of values for three risk components. Out of the possible combinations, the most risky in terms or rapid diversification and spillover potential is high phylogenetic diversity and low viral sharing,<span class="citation" data-cites="Gomulkiewicz2000HotSpo"><sup>58</sup></span> in that this allows multiple independent host-virus coevolutionary dynamics to take place in the same location. In the colorimetric space, this correspond to yellow – because the HSV space is more amenable to calculations for feature extraction,<span class="citation" data-cites="Keke2010StuSki"><sup>59</sup></span> we measured the risk level by calculating the angular distance of the hue of each pixel to a reference value of 60 (yellow), and weighted this risk level by the value component. Specifically, given a pixel with colorimetric coordinates <span class="math inline">(<em>h</em>, <em>s</em>, <em>v</em>)</span>, its ranged weighted risk value is</p>
            <p><span class="math display">$$
            v\times\left[1-\frac{\left|\text{atan}\left(\text{cos}(\text{rad}(h)), \text{sin}(\text{rad}(h))\right) - X\right|}{2\pi}\right]\,,
            $$</span></p>
            <p>where X is <span class="math inline">atan(cos(rad(60)),sin(rad(60)))</span>, a constant approximately equal to <span class="math inline">0.5235</span>.</p>
            <h2 id="viral-phylogeography-and-evolutionary-diversification">Viral phylogeography and evolutionary diversification</h2>
            <p>To next represent phylogeography of betacoronaviruses in bats, we aggregated and analyzed betacoronavirus sequence data. We used the following query to pull all <em>Betacoronavirus</em> sequence data from the GenBank Nucleotide database except SARS-CoV-2; (“Betacoronavirus”[Organism] OR betacoronavirus[All Fields]) NOT (“Severe acute respiratory syndrome coronavirus 2”[Organism] OR sars-cov-2[All Fields]). We added a single representative sequence for SARS-CoV-2 and manually curated to remove sequences without the RNA-dependent RNA polymerase (RdRp) sequence or that contained words indicating recombinant or laboratory strains including “patent”, “mutant”, “GFP”, and “recombinant”. We filtered over-represented taxa including betacoronavirus 1, hCoV-OC43, Middle East respiratory syndrome coronavirus, Murine hepatitis virus, and hCoV-HKU1. Curated betacoronavirus RdRp sequences were then aligned using MAFFT<span class="citation" data-cites="Katoh2013MafMul"><sup>60</sup></span> v1.4.0 (Algorithm FFT-NS-2, Scoring matrix 200PAM / k=2, gap open penalty 1.53m offset value 0.123) and a maximum likelihood tree reconstructed in IQ-TREE<span class="citation" data-cites="Nguyen2015IqtFas"><sup>61</sup></span> v1.6.12 with ModelFinder<span class="citation" data-cites="Kalyaanamoorthy2017ModFas"><sup>62</sup></span> ultrafast bootstrap approximation<span class="citation" data-cites="Hoang2018UfbImp"><sup>63</sup></span> with a general time reversible model with empirical base frequencies and the 5-discrete-rate-category FreeRaye model of nucleotide substitution (GTR+F+R5).</p>
            <p>We first tested the hypothesis that hotspots of viral diversification would track hotspots of bat diversification. To do so, we plotted the number of known bat hosts (specifically only those included in the phylogeny, so there was a 1:1 correspondence between data sources) against the “mean evolutionary distinctiveness” of the associated viruses. To calculate this, we derived the fair proportions evolutionary distinctiveness<span class="citation" data-cites="Isaac2007MamEdg"><sup>64</sup></span> for each of the viruses in the tree, then averaged these at the bat species level, projected these values onto their geographic distributions, and averaged across every bat found in a given pixel. As such, this can be thought of as a map of the mean evolutionary distinctiveness of the known viral community believed to be associated with a particular subset of bats present.</p>
            <h2 id="co-distribution-of-hosts-and-viral-hotspots">Co-distribution of hosts and viral hotspots</h2>
            <p>Subsequently, we tested the hypothesis that the biogeography of bat betacoronaviruses should track the biogeography of their hosts. To test this idea, we loosely adapted a method from,<span class="citation" data-cites="Kreft2007GloPat Kreft2010FraDel"><sup>65,66</sup></span> who proposed a phylogenetic method for the delineation of animal biogeographic regions. In their original method, a distance matrix - where each row or column represents a geographic raster’s grid cell, and the dissimilarity values are the “beta diversity similarity” of their community assemble - undergoes non-metric multidimensional scaling (NMDS); the first two axes of the NMDS are projected geographically using a four-color bivariate map. Here, we build on this idea with an entirely novel methodology. First, we measure the phylogenetic distance between the different viruses in the betacoronaviruses tree by using the cophenetic function in <code>ape</code>;<span class="citation" data-cites="Paradis2019ApeEnv"><sup>67</sup></span> subsequently, we take a principal components analysis of that distance matrix (readily interchangeable for NMDS in this case) to project the viral tree into an n-dimensional space. We then take the first two principal components and, as with the evolutionary distinctiveness analysis, aggregated these to a mean host value and projected them using a four-color bivariate map.</p>
            <h1 class="unnumbered" id="references">References</h1>
            <div id="refs" class="references csl-bib-body" data-line-spacing="2" role="doc-bibliography">
            <div id="ref-Plowright2017PatZoo" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">1. </div><div class="csl-right-inline">Plowright, R. K. <em>et al.</em> Pathways to zoonotic spillover. <em>Nature Reviews Microbiology</em> <strong>15</strong>, 502–510 (2017).</div>
            </div>
            <div id="ref-Anthony2017GloPat" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">2. </div><div class="csl-right-inline">Anthony, S. J. <em>et al.</em> Global patterns in coronavirus diversity. <em>Virus Evolution</em> <strong>3</strong>, (2017).</div>
            </div>
            <div id="ref-Ruiz-Aravena2022EcoEvo" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">3. </div><div class="csl-right-inline">Ruiz-Aravena, M. <em>et al.</em> Ecology, evolution and spillover of coronaviruses from bats. <em>Nature Reviews Microbiology</em> <strong>20</strong>, 299–314 (2022).</div>
            </div>
            <div id="ref-Agosta2010HowSpe" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">4. </div><div class="csl-right-inline">Agosta, S. J., Janz, N. &amp; Brooks, D. R. How specialists can be generalists: Resolving the <span>‘parasite paradox’</span> and implications for emerging infectious disease. <em>Zoologia (Curitiba)</em> <strong>27</strong>, 151–162 (2010).</div>
            </div>
            <div id="ref-Power2004PatSpi" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">5. </div><div class="csl-right-inline">Power, A. G. &amp; Mitchell, C. E. Pathogen Spillover in Disease Epidemics. <em>The American Naturalist</em> <strong>164</strong>, S79–S89 (2004).</div>
            </div>
            <div id="ref-Geoghegan2017ComAna" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">6. </div><div class="csl-right-inline">Geoghegan, J. L., Duchêne, S. &amp; Holmes, E. C. Comparative analysis estimates the relative frequencies of co-divergence and cross-species transmission within viral families. <em>PLOS Pathogens</em> <strong>13</strong>, e1006215 (2017).</div>
            </div>
            <div id="ref-Thompson2005GeoMos" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">7. </div><div class="csl-right-inline">Thompson, J. N. <em>The Geographic Mosaic of Coevolution</em>. (University Of Chicago Press, 2005).</div>
            </div>
            <div id="ref-Thompson1994CoePro" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">8. </div><div class="csl-right-inline">Thompson, J. N. <em>The Coevolutionary Process</em>. (University of Chicago Press, 1994).</div>
            </div>
            <div id="ref-Janzen1980WheIt" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">9. </div><div class="csl-right-inline">Janzen, D. H. When is it Coevolution? <em>Evolution</em> <strong>34</strong>, 611–612 (1980).</div>
            </div>
            <div id="ref-Price2002MacThe" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">10. </div><div class="csl-right-inline">Price, P. W. <em>Macroevolutionary Theory on Macroecological Patterns</em>. (Cambridge University Press, 2002). doi:<a href="https://doi.org/10.1017/CBO9780511615030">10.1017/CBO9780511615030</a>.</div>
            </div>
            <div id="ref-Gomulkiewicz2007DosDon" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">11. </div><div class="csl-right-inline">Gomulkiewicz, R. <em>et al.</em> Dos and don’ts of testing the geographic mosaic theory of coevolution. <em>Heredity</em> <strong>98</strong>, 249–258 (2007).</div>
            </div>
            <div id="ref-Thrall2007CoeSym" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">12. </div><div class="csl-right-inline">Thrall, P. H., Hochberg, M. E., Burdon, J. J. &amp; Bever, J. D. Coevolution of symbiotic mutualists and parasites in a community context. <em>Trends in Ecology &amp; Evolution</em> <strong>22</strong>, 120–126 (2007).</div>
            </div>
            <div id="ref-Olival2017HosVir" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">13. </div><div class="csl-right-inline">Olival, K. J. <em>et al.</em> Host and viral traits predict zoonotic spillover from mammals. <em>Nature</em> <strong>546</strong>, 646–650 (2017).</div>
            </div>
            <div id="ref-Becker2022OptPre" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">14. </div><div class="csl-right-inline">Becker, D. J. <em>et al.</em> Optimising predictive models to prioritise viral discovery in zoonotic reservoirs. <em>The Lancet Microbe</em> (2022) doi:<a href="https://doi.org/10.1016/S2666-5247(21)00245-7">10.1016/S2666-5247(21)00245-7</a>.</div>
            </div>
            <div id="ref-Allen2017GloHot" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">15. </div><div class="csl-right-inline">Allen, T. <em>et al.</em> Global hotspots and correlates of emerging zoonotic diseases. <em>Nature Communications</em> <strong>8</strong>, (2017).</div>
            </div>
            <div id="ref-Cohen2022SamStr" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">16. </div><div class="csl-right-inline">Cohen, L. E., Fagre, A. C., Chen, B., Carlson, C. J. &amp; Becker, D. J. Sampling strategies and pre-pandemic surveillance gaps for bat coronaviruses. 2022.06.15.496296 (2022) doi:<a href="https://doi.org/10.1101/2022.06.15.496296">10.1101/2022.06.15.496296</a>.</div>
            </div>
            <div id="ref-Latinne2020OriCro" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">17. </div><div class="csl-right-inline">Latinne, A. <em>et al.</em> Origin and cross-species transmission of bat coronaviruses in China. <em>bioRxiv: The Preprint Server for Biology</em> 2020.05.31.116061 (2020) doi:<a href="https://doi.org/10.1101/2020.05.31.116061">10.1101/2020.05.31.116061</a>.</div>
            </div>
            <div id="ref-VanBrussel2022ZooDis" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">18. </div><div class="csl-right-inline">Van Brussel, K. &amp; Holmes, E. C. Zoonotic disease and virome diversity in bats. <em>Current Opinion in Virology</em> <strong>52</strong>, 192–202 (2022).</div>
            </div>
            <div id="ref-Letko2020BatVir" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">19. </div><div class="csl-right-inline">Letko, M., Seifert, S. N., Olival, K. J., Plowright, R. K. &amp; Munster, V. J. Bat-borne virus diversity, spillover and emergence. <em>Nature Reviews Microbiology</em> <strong>18</strong>, 461–471 (2020).</div>
            </div>
            <div id="ref-Banerjee2020NovIns" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">20. </div><div class="csl-right-inline">Banerjee, A. <em>et al.</em> Novel Insights Into Immune Systems of Bats. <em>Frontiers in Immunology</em> <strong>11</strong>, 26 (2020).</div>
            </div>
            <div id="ref-Gorbunova2020WorGoe" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">21. </div><div class="csl-right-inline">Gorbunova, V., Seluanov, A. &amp; Kennedy, B. K. The World Goes Bats: Living Longer and Tolerating Viruses. <em>Cell Metabolism</em> <strong>32</strong>, 31–43 (2020).</div>
            </div>
            <div id="ref-Irving2021LesHos" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">22. </div><div class="csl-right-inline">Irving, A. T., Ahn, M., Goh, G., Anderson, D. E. &amp; Wang, L.-F. Lessons from the host defences of bats, a unique viral reservoir. <em>Nature</em> <strong>589</strong>, 363–370 (2021).</div>
            </div>
            <div id="ref-Jebb2020SixRef" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">23. </div><div class="csl-right-inline">Jebb, D. <em>et al.</em> Six reference-quality genomes reveal evolution of bat adaptations. <em>Nature</em> <strong>583</strong>, 578–584 (2020).</div>
            </div>
            <div id="ref-MorenoSantillan2021LarGen" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">24. </div><div class="csl-right-inline">Moreno Santillán, D. D. <em>et al.</em> Large-scale genome sampling reveals unique immunity and metabolic adaptations in bats. <em>Molecular Ecology</em> mec.16027 (2021) doi:<a href="https://doi.org/10.1111/mec.16027">10.1111/mec.16027</a>.</div>
            </div>
            <div id="ref-Muylaert2022PreFut" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">25. </div><div class="csl-right-inline">Muylaert, R. L. <em>et al.</em> Present and future distribution of bat hosts of sarbecoviruses: Implications for conservation and public health. <em>Proceedings of the Royal Society B: Biological Sciences</em> <strong>289</strong>, 20220397 (2022).</div>
            </div>
            <div id="ref-Wang2018SerEvi" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">26. </div><div class="csl-right-inline">Wang, N. <em>et al.</em> Serological Evidence of Bat SARS-Related Coronavirus Infection in Humans, China. <em>Virologica Sinica</em> <strong>33</strong>, 104–107 (2018).</div>
            </div>
            <div id="ref-Ramshaw2019DatGeo" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">27. </div><div class="csl-right-inline">Ramshaw, R. E. <em>et al.</em> A database of geopositioned Middle East Respiratory Syndrome Coronavirus occurrences. <em>Scientific Data</em> <strong>6</strong>, 318 (2019).</div>
            </div>
            <div id="ref-Lednicky2021IsoNov" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">28. </div><div class="csl-right-inline">Lednicky, J. A. <em>et al.</em> Isolation of a Novel Recombinant Canine Coronavirus from a Visitor to Haiti: Further Evidence of Transmission of Coronaviruses of Zoonotic Origin to Humans. <em>Clinical Infectious Diseases: An Official Publication of the Infectious Diseases Society of America</em> ciab924 (2021) doi:<a href="https://doi.org/10.1093/cid/ciab924">10.1093/cid/ciab924</a>.</div>
            </div>
            <div id="ref-Vlasova2022AniAlp" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">29. </div><div class="csl-right-inline">Vlasova, A. N. <em>et al.</em> Animal alphacoronaviruses found in human patients with acute respiratory illness in different countries. <em>Emerging Microbes &amp; Infections</em> <strong>11</strong>, 699–702 (2022).</div>
            </div>
            <div id="ref-Shi2014DeeDiv" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">30. </div><div class="csl-right-inline">Shi, J. J. <em>et al.</em> A Deep Divergence Time between Sister Species of Eidolon (Pteropodidae) with Evidence for Widespread Panmixia. <em>Acta Chiropterologica</em> <strong>16</strong>, 279–292 (2014).</div>
            </div>
            <div id="ref-Kettenburg2022FulGen" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">31. </div><div class="csl-right-inline">Kettenburg, G. <em>et al.</em> Full Genome Nobecovirus Sequences From Malagasy Fruit Bats Define a Unique Evolutionary History for This Coronavirus Clade. <em>Frontiers in Public Health</em> <strong>10</strong>, (2022).</div>
            </div>
            <div id="ref-Hazarie2021IntPop" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">32. </div><div class="csl-right-inline">Hazarie, S., Soriano-Paños, D., Arenas, A., Gómez-Gardeñes, J. &amp; Ghoshal, G. Interplay between population density and mobility in determining the spread of epidemics in cities. <em>Communications Physics</em> <strong>4</strong>, 191 (2021).</div>
            </div>
            <div id="ref-Xu2004EpiClu" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">33. </div><div class="csl-right-inline">Xu, R.-H. <em>et al.</em> Epidemiologic Clues to SARS Origin in China. <em>Emerging Infectious Diseases</em> <strong>10</strong>, 1030–1037 (2004).</div>
            </div>
            <div id="ref-Peixoto2018SynEco" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">34. </div><div class="csl-right-inline">Peixoto, F. P., Braga, P. H. P. &amp; Mendes, P. A synthesis of ecological and evolutionary determinants of bat diversity across spatial scales. <em>BMC ecology</em> <strong>18</strong>, 18 (2018).</div>
            </div>
            <div id="ref-Simmons2020BatSpe" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">35. </div><div class="csl-right-inline">Simmons, N. B. &amp; Cirranello, A. L. Bat Species of the World: A taxonomic and geographic database. <a href="https://batnames.org/">https://batnames.org/</a> (2020).</div>
            </div>
            <div id="ref-Kasso2013EcoEco" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">36. </div><div class="csl-right-inline">Kasso, M. &amp; Balakrishnan, M. Ecological and Economic Importance of Bats (Order Chiroptera). <em>ISRN Biodiversity</em> <strong>2013</strong>, e187415 (2013).</div>
            </div>
            <div id="ref-Mello2011MisPar" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">37. </div><div class="csl-right-inline">Mello, M. A. R. <em>et al.</em> The Missing Part of Seed Dispersal Networks: Structure and Robustness of Bat-Fruit Interactions. <em>PLOS ONE</em> <strong>6</strong>, e17395 (2011).</div>
            </div>
            <div id="ref-Voigt2016BatAnt" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">38. </div><div class="csl-right-inline"><em>Bats in the Anthropocene: Conservation of Bats in a Changing World</em>. (Springer International Publishing, 2016). doi:<a href="https://doi.org/10.1007/978-3-319-25220-9">10.1007/978-3-319-25220-9</a>.</div>
            </div>
            <div id="ref-Williams-Guillen2008BatLim" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">39. </div><div class="csl-right-inline">Williams-Guillén, K., Perfecto, I. &amp; Vandermeer, J. Bats Limit Insects in a Neotropical Agroforestry System. <em>Science</em> <strong>320</strong>, 70–70 (2008).</div>
            </div>
            <div id="ref-Gonsalves2013MosCon" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">40. </div><div class="csl-right-inline">Gonsalves, L., Bicknell, B., Law, B., Webb, C. &amp; Monamy, V. Mosquito Consumption by Insectivorous Bats: Does Size Matter? <em>PLOS ONE</em> <strong>8</strong>, e77183 (2013).</div>
            </div>
            <div id="ref-Gonsalves2013MosInf" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">41. </div><div class="csl-right-inline">Gonsalves, L., Lamb, S., Webb, C., Law, B. &amp; Monamy, V. Do mosquitoes influence bat activity in coastal habitats? <em>Wildlife Research</em> <strong>40</strong>, 10–24 (2013).</div>
            </div>
            <div id="ref-Wu2021ComSur" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">42. </div><div class="csl-right-inline">Wu, Z. <em>et al.</em> <em>A comprehensive survey of bat sarbecoviruses across China for the origin tracing of SARS-CoV and SARS-CoV-2</em>. (2021) doi:<a href="https://doi.org/10.21203/rs.3.rs-885194/v1">10.21203/rs.3.rs-885194/v1</a>.</div>
            </div>
            <div id="ref-Carlson2022CliCha" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">43. </div><div class="csl-right-inline">Carlson, C. J. <em>et al.</em> Climate change increases cross-species viral transmission risk. <em>Nature</em> 1–1 (2022) doi:<a href="https://doi.org/10.1038/s41586-022-04788-w">10.1038/s41586-022-04788-w</a>.</div>
            </div>
            <div id="ref-Alves2018GeoVar" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">44. </div><div class="csl-right-inline">Alves, D. M. C. C., Diniz-Filho, J. A. F., da Silva e Souza, K., Gouveia, S. F. &amp; Villalobos, F. Geographic variation in the relationship between large-scale environmental determinants and bat species richness. <em>Basic and Applied Ecology</em> <strong>27</strong>, 1–8 (2018).</div>
            </div>
            <div id="ref-Treitler2016EffLoc" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">45. </div><div class="csl-right-inline">Treitler, J. T., Heim, O., Tschapka, M. &amp; Jung, K. The effect of local land use and loss of forests on bats and nocturnal insects. <em>Ecology and Evolution</em> <strong>6</strong>, 4289–4297 (2016).</div>
            </div>
            <div id="ref-Johnson2020GloShi" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">46. </div><div class="csl-right-inline">Johnson, C. K. <em>et al.</em> Global shifts in mammalian population trends reveal key predictors of virus spillover risk. <em>Proceedings of the Royal Society B: Biological Sciences</em> <strong>287</strong>, 20192736 (2020).</div>
            </div>
            <div id="ref-Gryseels2017WheVir" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">47. </div><div class="csl-right-inline">Gryseels, S. <em>et al.</em> When Viruses Don’t Go Viral: The Importance of Host Phylogeographic Structure in the Spatial Spread of Arenaviruses. <em>PLOS Pathogens</em> <strong>13</strong>, e1006073 (2017).</div>
            </div>
            <div id="ref-Amman2011InvRol" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">48. </div><div class="csl-right-inline">Amman, B. R. <em>et al.</em> <em>Investigating the role of bats in emerging zoonoses: Balancing ecology, conservation and public health interest</em>. (FAO, 2011).</div>
            </div>
            <div id="ref-IUCN2021IucRed" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">49. </div><div class="csl-right-inline">IUCN. The IUCN Red List of Threatened Species. <a href="https://www.iucnredlist.org">https://www.iucnredlist.org</a> (2021).</div>
            </div>
            <div id="ref-RouaultEven2022GdaOgr" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">50. </div><div class="csl-right-inline">Rouault, E. <em>et al.</em> <em>GDAL/OGR Geospatial Data Abstraction software Library</em>. (Zenodo, 2022). doi:<a href="https://doi.org/10.5281/ZENODO.5884351">10.5281/ZENODO.5884351</a>.</div>
            </div>
            <div id="ref-Faith1992ConEva" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">51. </div><div class="csl-right-inline">Faith, D. P. Conservation evaluation and phylogenetic diversity. <em>Biological Conservation</em> <strong>61</strong>, 1–10 (1992).</div>
            </div>
            <div id="ref-Upham2019InfMam" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">52. </div><div class="csl-right-inline">Upham, N. S., Esselstyn, J. A. &amp; Jetz, W. Inferring the mammal tree: Species-level sets of phylogenies for questions in ecology, evolution, and conservation. <em>PLOS Biology</em> <strong>17</strong>, e3000494 (2019).</div>
            </div>
            <div id="ref-Legendre2013BetDiv" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">53. </div><div class="csl-right-inline">Legendre, P. &amp; De Cáceres, M. Beta diversity as the variance of community data: Dissimilarity coefficients and partitioning. <em>Ecology Letters</em> <strong>16</strong>, 951–963 (2013).</div>
            </div>
            <div id="ref-Legendre2019SpaTem" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">54. </div><div class="csl-right-inline">Legendre, P. &amp; Condit, R. Spatial and temporal analysis of beta diversity in the Barro Colorado Island forest dynamics plot, Panama. <em>Forest Ecosystems</em> <strong>6</strong>, 7 (2019).</div>
            </div>
            <div id="ref-Dansereau2022EvaEco" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">55. </div><div class="csl-right-inline">Dansereau, G., Legendre, P. &amp; Poisot, T. Evaluating ecological uniqueness over broad spatial extents using species distribution modelling. <em>Oikos</em> <strong>n/a</strong>, e09063 (2022).</div>
            </div>
            <div id="ref-Albery2020PreGlo" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">56. </div><div class="csl-right-inline">Albery, G. F., Eskew, E. A., Ross, N. &amp; Olival, K. J. Predicting the global mammalian viral sharing network using phylogeography. <em>Nature Communications</em> <strong>11</strong>, 2260 (2020).</div>
            </div>
            <div id="ref-Seekell2018GeoLak" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">57. </div><div class="csl-right-inline">Seekell, D. A., Lapierre, J.-F. &amp; Cheruvelil, K. S. A geography of lake carbon cycling. <em>Limnology and Oceanography Letters</em> <strong>3</strong>, 49–56 (2018).</div>
            </div>
            <div id="ref-Gomulkiewicz2000HotSpo" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">58. </div><div class="csl-right-inline">Gomulkiewicz, R., Thompson, J. N., Holt, R. D., Nuismer, S. L. &amp; Hochberg, M. E. Hot Spots, Cold Spots, and the Geographic Mosaic Theory of Coevolution. <em>The American Naturalist</em> <strong>156</strong>, 156–174 (2000).</div>
            </div>
            <div id="ref-Keke2010StuSki" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">59. </div><div class="csl-right-inline">Keke, S., Peng, Z. &amp; Guohui, L. Study on skin color image segmentation used by Fuzzy-c-means arithmetic. in <em>2010 Seventh International Conference on Fuzzy Systems and Knowledge Discovery</em> vol. 2 612–615 (2010).</div>
            </div>
            <div id="ref-Katoh2013MafMul" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">60. </div><div class="csl-right-inline">Katoh, K. &amp; Standley, D. M. MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability. <em>Molecular Biology and Evolution</em> <strong>30</strong>, 772–780 (2013).</div>
            </div>
            <div id="ref-Nguyen2015IqtFas" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">61. </div><div class="csl-right-inline">Nguyen, L.-T., Schmidt, H. A., von Haeseler, A. &amp; Minh, B. Q. IQ-TREE: A Fast and Effective Stochastic Algorithm for Estimating Maximum-Likelihood Phylogenies. <em>Molecular Biology and Evolution</em> <strong>32</strong>, 268–274 (2015).</div>
            </div>
            <div id="ref-Kalyaanamoorthy2017ModFas" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">62. </div><div class="csl-right-inline">Kalyaanamoorthy, S., Minh, B. Q., Wong, T. K. F., von Haeseler, A. &amp; Jermiin, L. S. ModelFinder: Fast model selection for accurate phylogenetic estimates. <em>Nature Methods</em> <strong>14</strong>, 587–589 (2017).</div>
            </div>
            <div id="ref-Hoang2018UfbImp" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">63. </div><div class="csl-right-inline">Hoang, D. T., Chernomor, O., von Haeseler, A., Minh, B. Q. &amp; Vinh, L. S. UFBoot2: Improving the Ultrafast Bootstrap Approximation. <em>Molecular Biology and Evolution</em> <strong>35</strong>, 518–522 (2018).</div>
            </div>
            <div id="ref-Isaac2007MamEdg" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">64. </div><div class="csl-right-inline">Isaac, N. J. B., Turvey, S. T., Collen, B., Waterman, C. &amp; Baillie, J. E. M. Mammals on the EDGE: Conservation Priorities Based on Threat and Phylogeny. <em>PLOS ONE</em> <strong>2</strong>, e296 (2007).</div>
            </div>
            <div id="ref-Kreft2007GloPat" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">65. </div><div class="csl-right-inline">Kreft, H. &amp; Jetz, W. Global patterns and determinants of vascular plant diversity. <em>Proceedings of the National Academy of Sciences</em> <strong>104</strong>, 5925–5930 (2007).</div>
            </div>
            <div id="ref-Kreft2010FraDel" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">66. </div><div class="csl-right-inline">Kreft, H. &amp; Jetz, W. A framework for delineating biogeographical regions based on species distributions. <em>Journal of Biogeography</em> <strong>37</strong>, 2029–2053 (2010).</div>
            </div>
            <div id="ref-Paradis2019ApeEnv" class="csl-entry" role="doc-biblioentry">
            <div class="csl-left-margin">67. </div><div class="csl-right-inline">Paradis, E. &amp; Schliep, K. Ape 5.0: An environment for modern phylogenetics and evolutionary analyses in R. <em>Bioinformatics</em> <strong>35</strong>, 526–528 (2019).</div>
            </div>
            </div>
        </article>
    </main>

    <footer>
        <div class="container">
            <h2>Affiliations</h2> 
            <sup>1</sup>&nbsp;Université de Montréal; <sup>2</sup>&nbsp;Québec Centre for Biodiversity Sciences; <sup>3</sup>&nbsp;Molecular Epidemiology and Public Health Laboratory, School of Veterinary Science, Massey University, New Zealand; <sup>4</sup>&nbsp;Paul G. Allen School for Global Health, Washington State University, Pullman, WA, United States; <sup>5</sup>&nbsp;Department of Biology, Georgetown University, Washington, DC, USA; <sup>6</sup>&nbsp;Department of Biology, University of Oklahoma, Norman, OK, USA; <sup>7</sup>&nbsp;Department of Biology, Georgetown University, Washington, DC,; <sup>8</sup>&nbsp;Center for Global Health Science and Security, Georgetown University Medical Center, Washington, DC, USA; <sup>9</sup>&nbsp;Department of Microbiology and Immunology, Georgetown University Medical Center, Washington, DC, USA
            <br />
            <sup>‡</sup>&nbsp;These authors contributed equally to the work
        </div>
    </footer>
    
</body>
</html>