%!TEX TS-program = xelatex
\documentclass[10pt,oneside]{article}
\usepackage[fontsize=9pt]{scrextend}

\usepackage[english]{babel}

\usepackage{amsmath,amssymb,amsfonts}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{stix2}
\usepackage[scaled]{helvet}
\usepackage[scaled]{inconsolata}

\usepackage{lastpage}

\usepackage{setspace}

\usepackage{ccicons}

\usepackage[hang,flushmargin]{footmisc}

\usepackage{geometry}

\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}

\usepackage{fancyhdr}
\renewcommand{\headrulewidth}{0pt}\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

\makeatletter
\newcounter{tableno}
\newenvironment{tablenos:no-prefix-table-caption}{
  \caption@ifcompatibility{}{
    \let\oldthetable\thetable
    \let\oldtheHtable\theHtable
    \renewcommand{\thetable}{tableno:\thetableno}
    \renewcommand{\theHtable}{tableno:\thetableno}
    \stepcounter{tableno}
    \captionsetup{labelformat=empty}
  }
}{
  \caption@ifcompatibility{}{
    \captionsetup{labelformat=default}
    \let\thetable\oldthetable
    \let\theHtable\oldtheHtable
    \addtocounter{table}{-1}
  }
}
\makeatother

\usepackage{array}
\newcommand{\PreserveBackslash}[1]{\let\temp=\\#1\let\\=\temp}
\let\PBS=\PreserveBackslash

\usepackage[breaklinks=true]{hyperref}
\hypersetup{colorlinks,%
citecolor=blue,%
filecolor=blue,%
linkcolor=blue,%
urlcolor=blue}
\usepackage{url}

\usepackage{caption}
\setcounter{secnumdepth}{0}
\usepackage{cleveref}

\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth
\else\Gin@nat@width\fi}
\makeatother
\let\Oldincludegraphics\includegraphics
\renewcommand{\includegraphics}[1]{\Oldincludegraphics[width=\maxwidth]{#1}}

\usepackage{longtable}
\usepackage{booktabs}

\usepackage{color}
\usepackage{fancyvrb}
\newcommand{\VerbBar}{|}
\newcommand{\VERB}{\Verb[commandchars=\\\{\}]}
\DefineVerbatimEnvironment{Highlighting}{Verbatim}{commandchars=\\\{\}}
% Add ',fontsize=\small' for more characters per line
\usepackage{framed}
\definecolor{shadecolor}{RGB}{248,248,248}
\newenvironment{Shaded}{\begin{snugshade}}{\end{snugshade}}
\newcommand{\KeywordTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\DataTypeTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{#1}}
\newcommand{\DecValTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\BaseNTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\FloatTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\ConstantTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\CharTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\SpecialCharTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\StringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\VerbatimStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\SpecialStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\ImportTok}[1]{#1}
\newcommand{\CommentTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\DocumentationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\AnnotationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\CommentVarTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\OtherTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{#1}}
\newcommand{\FunctionTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\VariableTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\ControlFlowTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\OperatorTok}[1]{\textcolor[rgb]{0.81,0.36,0.00}{\textbf{#1}}}
\newcommand{\BuiltInTok}[1]{#1}
\newcommand{\ExtensionTok}[1]{#1}
\newcommand{\PreprocessorTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\AttributeTok}[1]{\textcolor[rgb]{0.77,0.63,0.00}{#1}}
\newcommand{\RegionMarkerTok}[1]{#1}
\newcommand{\InformationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\WarningTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\AlertTok}[1]{\textcolor[rgb]{0.94,0.16,0.16}{#1}}
\newcommand{\ErrorTok}[1]{\textcolor[rgb]{0.64,0.00,0.00}{\textbf{#1}}}
\newcommand{\NormalTok}[1]{#1}

\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newenvironment{CSLReferences}[3] % #1 hanging-ident, #2 entry spacing
 {% don't indent paragraphs
  \setlength{\parindent}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1 \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces\fi
  % set entry spacing
  \ifnum #2 > 0
  \setlength{\parskip}{#2\baselineskip}
  \fi
 }%
 {}
\usepackage{calc} % for \widthof, \maxof
\newcommand{\CSLBlock}[1]{#1\hfill\break}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\maxof{\widthof{#1}}{\csllabelwidth}}{#1}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth}{#1}}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}\usepackage[table,dvipsnames]{xcolor}

\geometry{includemp,
            letterpaper,
            top=2.4cm,
            bottom=2.4cm,
            left=1.0cm,
            right=1.0cm,
            marginparwidth=5cm,
            marginparsep=1.0cm}

\usepackage[singlelinecheck=off]{caption}

\captionsetup{
  font={small},
  labelfont={bf},
  format=plain,
  labelsep=quad
}

\usepackage{floatrow}

\floatsetup[figure]{margins=hangright,
              facing=no,
              capposition=beside,
              capbesideposition={center,outside},
              floatwidth=\textwidth}

% \floatsetup[table]{margins=hangright,
%              facing=no,
%              capposition=beside,
%              capbesideposition={center,outside},
%              floatwidth=\textwidth}

\pagestyle{plain}

\setcounter{secnumdepth}{5}

\usepackage{titlesec}

\titleformat{\section}[block]
{\normalfont\large\sffamily}
{\thesection}{.5em}{\titlerule\\[.8ex]\bfseries}

\titleformat{\subsection}[runin]
{\normalfont\fontseries{b}\selectfont\filright\sffamily}
{\thesubsection.}{.5em}{}

\titleformat{\subsubsection}[runin]
{\normalfont\itshape\rmfamily\bfseries}{\thesubsubsection}{1em}{}

\fancypagestyle{firstpage}
{
   \fancyhf{}
   \renewcommand{\headrulewidth}{0pt}
   \fancyfoot[R]{\footnotesize\ccby}
   \fancyfoot[L]{\footnotesize\sffamily\today}
}

\fancypagestyle{normal}
{
  \fancyhf{}
  \fancyfoot[R]{\footnotesize\sffamily\thepage\ of \pageref*{LastPage}}
}

\usepackage{tikz}
\begin{document}
\tikz [remember picture, overlay] %
\node [shift={(-0.6in,1.1cm)},scale=0.2,opacity=0.4] at (current page.south east)[anchor=south east]{\includegraphics{logo}};%
\pagestyle{normal}
\thispagestyle{firstpage}

\newcommand{\colorRule}[3][black]{\textcolor[HTML]{#1}{\rule{#2}{#3}}}

\noindent {\LARGE \textbf{\textsf{The coevolutionary mosaic of
betacoronavirus emergence risk}}}

\medskip
\begin{flushleft}
{\small
%
Norma\,Forero Rocio Munoz%
%
\,\textsuperscript{1,2,‡}, %
Renata L.\,Muylaert%
%
\,\textsuperscript{3}, %
Stephanie N.\,Seifert%
%
\,\textsuperscript{4}, %
Gregory F.\,Albery%
%
\,\textsuperscript{5}, %
Daniel J.\,Becker%
%
\,\textsuperscript{6}, %
Colin J.\,Carlson%
%
\,\textsuperscript{7,8,9,‡}, %
\href{https://orcid.org/0000-0002-0735-5184}{Timothée\,Poisot}%
%
\,\textsuperscript{1,2,‡}
\vskip 1em
\textsuperscript{1}\,Université de Montréal; \textsuperscript{2}\,Québec
Centre for Biodiversity Sciences; \textsuperscript{3}\,Molecular
Epidemiology and Public Health Laboratory, School of Veterinary Science,
Massey University, New Zealand; \textsuperscript{4}\,Paul G. Allen
School for Global Health, Washington State University, Pullman, WA,
United States; \textsuperscript{5}\,Department of Biology, Georgetown
University, Washington, DC, USA; \textsuperscript{6}\,Department of
Biology, University of Oklahoma, Norman, OK,
USA; \textsuperscript{7}\,Department of Biology, Georgetown University,
Washington, DC,; \textsuperscript{8}\,Center for Global Health Science
and Security, Georgetown University Medical Center, Washington, DC,
USA; \textsuperscript{9}\,Department of Microbiology and Immunology,
Georgetown University Medical Center, Washington, DC, USA\\
\textsuperscript{‡}\,These authors contributed equally to the work\\
\vskip 1em
\textbf{Correspondance to:}\\
Timothée Poisot --- \texttt{timothee.poisot@umontreal.ca}\\
}
\end{flushleft}

\vskip 2em
\makebox[0pt][l]{\colorRule[CCCCCC]{2.0\textwidth}{0.5pt}}
\vskip 2em
\noindent

\marginpar{\vskip 1em\flushright
{\small{\bfseries Keywords}:\par
bats\\betacoronavirus\\disease ecology\\geographic mosaic theory of
coevolution\\phylogenetic diversity\\viral sharing\\SARS-CoV-2\\}
}


Driven by the need to understand the ecological factors involved in the
emergence of \emph{Betacoronavirus} (the genus causing the SARS, MERS,
and COVID-19 in human) through bat hosts, we develop an approach to the
assesment of spillover risk based on the Geographic Mosaic Theory of
Coevolution. In doing so, we provide a global mapping of the spillover
risk posed by betacoronaviruses, reflecting the fact that this risk is
best understood through the multi-faceted prism of ecological and
evolutionary mechanisms. Our framework reveals that host richness alone,
although a component of viral hazard, is not a sufficiently integrative
predictor of risk. We offer alternative insights based on viral sharing,
host compositional uniqueness, and host phylogenetic diversity and
phylogeographic regions. By comparing our aggregated measure of risk to
a proxy for human density, namely the proportion of each pixel that is
covered by urban or built land, we provide a synthetic risk map,
allowing the identification of hotspots where the bat-betacoronavirus
interaction network may facilitate the emergence of novel viruses and
their spillover into human populations.




\vskip 2em
\makebox[0pt][l]{\colorRule[CCCCCC]{2.0\textwidth}{0.5pt}}
\vskip 2em

Disease emergence is complex, and is driven not only by animal-human
contact, but also by the underlying evolutionary dynamics in viral
reservoirs.\textsuperscript{1} Although host richness is often used as a
superficial proxy for spillover risk,\textsuperscript{2,3} these
approaches oversimplify the relevant interspecific heterogeneity in
immunology, behavior, and other traits, and therefore overlook unique
host pools that allow for the rapid evolution of highly divergent
viruses.\textsuperscript{4} In the case of generalist pathogens like
betacoronaviruses, there is conceptual and empirical support to the idea
that these community-level mechanisms are even more
important,\textsuperscript{5} particularly given that cross-species
transmission may, as a rule, structure viral evolution more than
co-divergence with hosts.\textsuperscript{6} This creates a disconnect
between coevolutionary theory (including empirical evidence from
virology) and most existing ecological frameworks for mapping spillover
risk.

The Geographic Mosaic Theory of Coevolution (GMTC) attempts to
explicitly connect microevolutionary dynamics to the macroecology and
biogeography of symbiotic interactions.\textsuperscript{7} The GMTC
posits that coevolutionary processes among pairs\textsuperscript{8} or
complexes\textsuperscript{9} of species are structured in space by the
rippling effects of abiotic conditions onto evolutionary mechanism,
giving rise to fragmented systems with different structure and
ecologically dynamics over large spatial extents.\textsuperscript{10}
The GMTC predicts a spatial fragmentation of coevolutionary dynamics
under the joint action of three processes:\textsuperscript{11}
coevolutionary hot- and coldspots, which appear when the intensity of
\emph{interaction} (in terms of reciprocal fitness consequences) varies
spatially; selection mosaics, wherein the intensity of \emph{selection}
varies across space, driven by both the biotic complexity of the
community (locally diverse hosts and viruses are more biotically
complex) and the local favorability of the
environment;\textsuperscript{12} and trait remixing, which occurs when
coevolutionary dynamics are driven by by the arrival (or departure) of
\emph{functional traits}, through changes in community composition due
to invasions, meta-community dynamics, and disperal.

Here, we apply the GMTC to explore and explain the global biogeography
of betacoronaviruses, the group that includes SARS-CoV, MERS-CoV, and
SARS-CoV-2. In their bat reservoirs, coronaviruses evolve through a mix
of host jumps, recombination among disparate lineages, and, to a lesser
degree, co-divergence with their hosts---\textsuperscript{2}a mix of
mechanisms that creates a complex and nonlinear relationship between
host diversity and viral emergence. Working from a recently published
database of bat hosts of betacoronaviruses, we develop the first global
maps of both host and virus evolutionary distinctiveness and
biogeographic regions for this system. Aiming to explain these patterns,
we develop a generalized framework for applying the GMTC to host-virus
interactions, with a specific emphasis on the potential to create
independent coevolutionary dynamics (and therefore spatial fragmentation
in risk) through heterogeneity. We develop a trivariate risk assessment
system that connects each GMTC mechanism to a quantifiable aspect of
host-virus interactions: (i) viral sharing rates in host communities,
representing the strength of potential interaction between viruses and
any one host (i.e., places where viruses undergo constant host switching
may be coevolutionary coldspots); (ii) the phylogenetic diversity of
hosts, as a proxy for variation in the immunological mechanisms that
antagonize viruses (i.e., the selection mosaic); and (iii) the local
uniqueness of the bat community, representing the potential for viruses
to be exposed to novel host traits (e.g., variation in receptor
sequences). Together, we argue that these can be used to identify and
map the evolutionary drivers that---in conjunction with transmission
processes (e.g., viral prevalence in reservoirs and animal-human contact
rates)--- determine disease emergence risk.

\hypertarget{host-richness-does-not-predict-virus-distinctiveness}{%
\section{Host richness does not predict virus
distinctiveness}\label{host-richness-does-not-predict-virus-distinctiveness}}

Bats, the second most diverse groups of mammals, are found worldwide and
serve as the main animal reservoir for different strains of
betacoronaviruses.\textsuperscript{13} This has attracted attention to
areas where high diversity of bats, and therefore presumably high
diversity of betacoronaviruses, can be an important issue for human
health.\textsuperscript{14,15} By overlaying the IUCN range maps for
confirmed bat hosts of betacoronaviruses (fig.~\ref{fig:richness}; top),
we see that the the main hotspots (both in terms of size and higher
values) of host richness are primarily South-Eastern Asia, parts of
Southern Europe, and to a lesser extent parts of Africa in the -25-0
range of latitudes. The description of host richness is an important
first step towards understanding risk, as previous
research\textsuperscript{2,16} states that locally diverse bat
communities could maintain more viruses and hence, a higher probability
of having a pathogen that could represent a risk for human health.

\begin{figure}
\hypertarget{fig:richness}{%
\centering
\includegraphics{figures/combined_richness.png}
\caption{Top panel: relative diversity of known bat hosts of
betacoronaviruses. This map shows that the region with the largest
number of possible hosts is South-Eastern Asia. Bottom panel: congruence
between the evolutionary distinctiveness of the hosts (grey to blue) and
the viruses (grey to red). By contrast to the richness map, this reveals
that South America has the most evolutionary distinct hosts \emph{and}
viruses, whereas South-Eastern Asia and the Rift Valley region have
mostly distinct viruses. This is congruent with know results about New
World bats being evolutionary distinct, and suggests that they similarly
have distinct viruses.}\label{fig:richness}
}
\end{figure}

Nevertheless, locally diverse and virus-rich bat communities could
represent an increased risk of spillover under climate change through
the creation of novel interactions,\textsuperscript{17} and therefore
the diversity of \emph{Betacoronavirus} strains should similarly be
ccounted for. In fig.~\ref{fig:richness} (bottom), we contrast the
evolutionary distinctiveness of bats and viruses -- this reveals a
slightly different portrait than bat richness alone. Chiropterans can be
classified, from a macro-evolutionary standpoint, as Yangochiroptera and
Yinpterochiroptera elsewhere.\textsuperscript{18,19} Specifically, we
would expect that the so-called ``New World'' group of bats, being more
evolutionary distinct, would also have evolutionary distinct viruses.
Indeed fig.~\ref{fig:richness} (bottom) reveals it to be the case, and
this region harbors a distinct bat-betacoronaviruses complex. This can
be explained by the fact that Yangochiroptera, although not limited to
the western hemisphere, contain the highly diverse adaptive radiation in
the Phyllostomidae,\textsuperscript{20} which is restricted to the
western hemisphere. By contrast, South-Eastern Asia has a lot of
non-evolutionary distinct bats, who nevertheless hosted
evolutionary-distinct viruses.

It is noteworthy that outside of South America, viral evolutionary
distinctiveness does not accurately track host diversity, with some
areas having over-distinct viruses (eastern China but, oddly, not the
rest of southeast Asia). There are a number of likely explanations.
First, given the richness of bats in southeast Asia, many
betacoronaviruses likely remain to be discovered in this region. Indeed,
global predictions highlight that southeast Asia is a likely hostpot of
unconfirmed hosts of betacoronaviruses,\textsuperscript{21} which would
likely result in additional viral discoveries. This idea is unsurprising
given the growing realization, especially since the emergence of
SARS-CoV-2, that unique lineages of similar viruses are widespread in
bats but still mostly undescribed. The most distinct
bats-betacoronaviruses complex is found in South America, a region with
a comparatively lower number of hosts; this matches with isolation
through host vicariance, and may highlight a different co-evolutionary
dynamic. Alternatively, this distinctiveness hostpot may be a product of
under-sampling: South-America is one of the places where the fewest
\emph{Betacoronavirus} sequences have been
discovered,\textsuperscript{2,22,23} resulting in sparser phylogenetic
tree, thereby artificially inflating distinctiveness. Adding more
viruses would bring the distinctiveness of known sequences down.
Finally, South-America is the range of Phyllostomidae, a group of bats
that underwent explosive diversification events,\textsuperscript{20}
which may drive the emergence of multiple viral lineages.

\hypertarget{the-phylogeographic-regions-of-hosts-and-their-viruses-overlap}{%
\section{The phylogeographic regions of hosts and their viruses
overlap}\label{the-phylogeographic-regions-of-hosts-and-their-viruses-overlap}}

Despite the difference in evolutionary distinctiveness globally, there
are reasons to expect that the phylogeography of bats and
betacoronaviruses should show some degree of
congruence.\textsuperscript{24,25} In particular, this should be the
case if viruses can circulate among hosts and co-evolve with local hosts
communities, making their evolutionary process more than a byproduct of
host evolution. High density of hosts sharing the same virus (albeit
possibly different strains) can drive or result from evolution of the
bat antiviral immune system, resulting in spatially distinct
immunological responses, as evidenced in several bat
species.\textsuperscript{26} Immune characteristics that allow bats to
be better adapted to infection by emerging
viruses,\textsuperscript{27,28} in addition to being hardcoded in their
genome,\textsuperscript{29} may be related to a wide variety of
diets,\textsuperscript{26,30} themselves likely to be driven by spatial
effects, especially at the local scale -- bats, indeed, occupy a variety
of environments, and therefore display a variety of adaptations to these
environments.\textsuperscript{31}

\begin{figure}
\hypertarget{fig:biogeo}{%
\centering
\includegraphics{figures/combined_biogeo.png}
\caption{Phylogeographic regions of bats (top) and viruses (bottom)
based on the joint analysis of their occurrence and phylogenetic
relatedness. The different colors show tendencies to separate alongside
the first two components of a PCoA. Note that the PCoA for the bats and
viruses are independent, and so cannot be compared directly -- that
being said, the regions can be compared across maps.}\label{fig:biogeo}
}
\end{figure}

In fig.~\ref{fig:biogeo}, we show a projection of the phylogeographic
signal of bats (top) and viruses (bottom) in space; the distinct
groupings (represented by different colors symbolizing positions in the
subspace formed by the first two axes of the PCoA) are essentially
equivalent between the two groups, and can be coarsely delineated as
southeast Asia, Eurasia above a northing of 25, and Africa and south
America. These results suggest that, although the evolutionary
distinctiveness of the bat-betacoronaviruses complex varies spatially,
the system shows an important degree of spatial consistency, with a
reduced number of bioregions. Available information describing the
spillover of zoonotic betacoronaviruses of bat origin where data was
available before and up through the COVID-19 pandemic puts spillover
events of SARS-CoV-2 in Wuhan, China; SARS-CoV in Guangdong, China based
on the presence of closest known viruses circulating in nature, and a
nearby location where serological (antibody) evidence has indicated
human exposure to SARS-like viruses;\textsuperscript{32} MERS-CoV in
Saudi Arabia based on index cases available from a recently-published
compendium of cases.\textsuperscript{33} For the latest event, most if
not all index cases are presumed to be camel-to-human transmission, and
the precise origin point (if it exists) of MERS-CoV in bats is
uncertain. Recent recombinant canine coronavirus spillover events in
Haiti\textsuperscript{34} and Europe\textsuperscript{35} are not
relevant here, as bats' involvement in these cycles of transmission have
been supposed to be non-existent. These index cases fall within
different phylogeographic bioregions (fig.~\ref{fig:biogeo}), which
further highlight the issue that different host-virus sub-systems may
lead to widespread emergence.

\hypertarget{coevolution-informed-emergence-risk-is-different-in-space}{%
\section{Coevolution-informed emergence risk is different in
space}\label{coevolution-informed-emergence-risk-is-different-in-space}}

As host richness, joint distinctiveness, or phylogeographic structure
suggest that the bat-betacoronaviruses complex is globally fragmented
enough to give rise to both different levels of risk (as evidenced by
the spatial location of spillover events) and different types of
co-evolutionary dynamics, we turn to the Geographic Mosaic Theory of
Coevolution to provide a measure of risk accounting for multiple
processes. In fig.~\ref{fig:trivariate}, we overlapped three components
of spillover risk: viral sharing, \emph{i.e.} the chance that two bats
will share viruses overall; Local Contribution to Beta Diversity,
\emph{i.e.} the fact that a bat community is compositionally unique
compared to the average compositional similarity across the entire
system; finally, host phylogenetic diversity, \emph{i.e.} how dispersed
the bats in a location are within the tree of life. This approach leads
to the definition of broad biogeographic regions of risk, where the same
color represents the same type of risk. By way of constrat to figures
fig.~\ref{fig:richness} and fig.~\ref{fig:biogeo}, these regions do not
necessarilly overlap with previous spatial partitions of the
bat-betacoronaviruses complex.

\begin{figure}
\hypertarget{fig:trivariate}{%
\centering
\includegraphics{figures/risk_trivariate.png}
\caption{Trivariate additive mapping of the components of risk in the
red/green/blue, where high virus sharing is encoded in the blue channel,
host phylogenetic diversity in the green channel, and compositional
uniqueness in the red channel. A pixel that would maximize all measures
(highest possible risk) would be a pure white (specifically RGB(1.0,
1.0. 1.0)), and a pixel with the lowest possible values would be pure
black (specifically RGB(0.0, 0.0, 0.0)). Therefore, lighter values (the
sum of the three channels gets closer to 3) indicate higher risk, and
the color indicates the proportional distribution of the factors making
up the total risk.}\label{fig:trivariate}
}
\end{figure}

From the perspective of spillover risk, the most important combination
of factors is a high phylogenetic diversity of hosts with low viral
sharing; this, essentially, means that very different betacoronaviruses
could co-exist within the same place. This is particularly the case
given that betacoronaviruses often evolve and even achieve host shifts
through recombination, which requires the co-occurrence of sufficiently
distinct viruses to be a major driver of emergence. In
fig.~\ref{fig:trivariate}, this corresponds to yellow to pale green
areas, which are essentially limited to South-Eastern Asia, and to some
part of Sub-Saharan Africa. Adopting a geographic mosaic theory
perspective on risk, other regions of the world are of lesser concern
(fig.~\ref{fig:risk}). Our risk decomposition does not account for viral
diversity or distinctiveness. The simple rationale behind it is that the
acquisition of viral data is rarely disconnected from the acquisition of
host data. There are more sources of information on hosts than on
viruses, allowing to develop a host-centric perspective on risk
(although this estimate would more accurate with viral traits related to
\emph{e.g.} ability to switch hosts or pathogenic potential). Areas with
high bat diversity and high turnover \emph{may} facilitate the
evolutionary radiation of viruses, matching previous findings that the
diversification of bat coronaviruses is driven largely by host shifts
(inter-genus or higher levels of cross-species transmission) and, to a
lesser degree, cospeciation and sharing, representing intra-genus
cross-species transmission.\textsuperscript{2} This diversification is
not an actual risk factor for spillover itself, but acts downstream of a
spillover event by increasing the random chance of the emergence of a
virus with the raw genomic components required for the potential to
infect humans.

\begin{figure}
\hypertarget{fig:risk}{%
\centering
\includegraphics{figures/risk_map.png}
\caption{Extraction of a measure of \emph{Betacoronavirus} spillover
risk from bat hosts based on the colorimetric space from
fig.~\ref{fig:trivariate}. The risk is a composite measure of the color
value and angular distance to the yellow hue, as defined in the methods,
ranged in the unit space. Based on this analyses, regions at high risk
of spillover are southeast Asia and Madagascar.}\label{fig:risk}
}
\end{figure}

From another perspective, areas of high host uniqueness and virus
sharing (red-to-pink) could provide hotspots of \emph{Betacoronavirus}
risk through mixing of unique viruses (via codivergence) and in turn
recombination. Under our framework, such a hotspot was identified in
Madagascar, where most bat species are endemic following evolutionary
divergence from sister species in both African and Asian
continents.\textsuperscript{36} Recent surveillance\textsuperscript{37}
has identified a novel \emph{Betacoronavirus} (in the subgenus
\emph{Nobecovirus}) in Madagascar-endemic pteropid bat species
(\emph{Pteropus rufus}, \emph{Rousettus madagascariensis}), emphasizing
strong proof of principle in model predictions.

\hypertarget{human-occupancy-drives-different-levels-of-effective-risk-globally}{%
\section{Human occupancy drives different levels of effective risk
globally}\label{human-occupancy-drives-different-levels-of-effective-risk-globally}}

Based on the previous result, we extracted the risk component from the
composite map (see Methods), to provide a single measure of risk varying
between 0 and 1. This measure is presented in fig.~\ref{fig:risk}. As
this map represents the potential risk, it must be weighed by the
potential for contacts with humans. As a proxy for this measure, we used
the proportion of build/urban land from the EarthEnv dataset: this is a
reasonable proxy for the density of humans per unit area, which
increases the probability of pathogen spread more
widely.\textsuperscript{38} Since human activity is required to amplify
the frequency of virus encounters and thus create areas of viral
amplification, mapping the potential risk against measures of land use
is required to generate a more actionable assessment of risk. This map
is presented in fig.~\ref{fig:compound}. Most of South America and
Europe are at comparatively lower risk, as although densely populated,
settlements tend to be in areas with lower potential risk. Regions like
Malaysia and the North coast of Australia have a high risk component,
but should represent a relatively lower effective risk due to low human
density. However, this mapping reveals that South-East Asia, the Indian
subcontinent, and parts of sub-Saharan Africa, are at high risk due to
the overlap between built areas and bat communities representing more
opportunities for cross-species transmission of betacoronaviruses. In
looking for the origins of SARS in China,\textsuperscript{39} present
serological evidence that strongest human-animal contact results in
higher risk of virus exposure, regardless of the animal species, but
that different types of contact had different impacts. Ideally,
finer-grained information about human activity (rather than human
presence through anthropisation) could allow to partition this risk
further, alebit at the cost of more hypotheses required to estimate the
amount of risk represented by each activity. Our map of purported high
risk/diversitifcation potential (Madagascar, South-America) overlay with
sampling gaps for \emph{Betacoronavirus},\textsuperscript{40} stressing
the need for spatially targeted monitoring and discovery.

\begin{figure}
\hypertarget{fig:compound}{%
\centering
\includegraphics{figures/risk_compounded.png}
\caption{Overlap of the percent of each pixel occupied by urbanized
structures, representing the degree of settlement, on the spillover risk
map (where the risk comes only from wildlife, and ignores multi-hosts
chains of transmissions including non-bats hosts). Darker pixels
correspond to more risk, in that the GMTC-derived risk of
fig.~\ref{fig:risk} is high \emph{and} the pixel is densely occupied by
human populations. This approach increases the relative risk of several
regions in Africa, and highlights the risk in India, southeast China,
and the Arabian peninsula where areas of high to moderate risk overlap
with areas of denser population.}\label{fig:compound}
}
\end{figure}

\hypertarget{conclusion}{%
\section{Conclusion}\label{conclusion}}

Bats are important reservoir hosts for different classes of
microorganisms, many of which a threat to human
health.\textsuperscript{24,25} Chiropterans emerged around 64 million
years ago and are one of the most diverse mammalian orders, with an
estimated richness of more than 1400 species.\textsuperscript{41,42}
They exhibit a broad variety of habitat use, behaviour, and feeding
strategies, putting them at key positions in the delivery and
provisioning of several ecosystem services, tied to important
ecosystem-derived benefits to human.\textsuperscript{43} For example,
bats are an essential component of many seed-dispersal
networks.\textsuperscript{44} Over two-thirds of bats are know to be
either obligate or facultative insectivores, therefore actively
contributing for agricultural pest control,\textsuperscript{45,46} and
vectors of pathogens that put a risk on human
health.\textsuperscript{47,48} Because bats are globally distributed and
have a long evolutionary history, phylogeographic and biogeographic
approaches are required to shed light on the contemporary distribution
of coevolutionary processes between bats and the pathogens they host.
Not all areas in which bats, viruses, and human are co-occuring are
facing a risk of spillover towards human populations, and the areas in
which this risk exist may not be facing risks of the same nature and
magnitude.

Here, we propose a simple freamework with broad explanatory power that
helps contextualize discoveries like highly divergent nobecoviruses in
Madagascar and the previously-neglected adaptive radiation of
sarbecoviruses outside of southern China and throughout southeast Asia.
In doing so, it advances ecological theory beyond the current state of
the art for global maps of emergence risk. For example, previous studies
that have used host richness as proxy have predicted a high diversity of
unsampled bat viruses,\textsuperscript{23} bat
coronaviruses,\textsuperscript{2} and even specifically
betacoronaviruses\textsuperscript{21} in both the Amazon and southeast
Asia. While we find that both regions are characterized by highly
divergent host and viral communities, our framework identifies key
differences between the regions. We find that Latin America is a hotspot
of both host and viral distinctiveness, suggesting that this branch of
the bat-betacoronavirus complex may be undergoing independent
evolutionary dynamics from the rest of the global pool, but with limited
potential for viral diversification--- a finding that is supported by
previous work indicating a higher rate of codivergence in Latin
America.\textsuperscript{2} In contrast, in southeast Asia, host
richness and viral distinctiveness are high but sharing is low; this
suggests a different type of evolutionary dynamics that could generate
high local diversity of viruses through host switching and viral
recombination (see \emph{e.g.},\textsuperscript{49} as well as the
discovery of recombinant viruses that share genetic material from both
the SARS-CoV and SARS-CoV-2 branches of the Sarbecovirus
lineage).\textsuperscript{50} Both of these regions are priority areas
for sampling, especially given predictions that they contain many bat
hosts of undiscovered betacoronaviruses.\textsuperscript{21,40} However,
both the evolutionary and ecological aspects of emergence risk are
likely higher in southeast Asia---a fact that will only become more
relevant, as bats track shifting climates and exchange viruses with
other species, creating a hotspot of cross-species transmission unique
to the region.\textsuperscript{17}

The diversity and diversification potential of bats responds to
anthropogenic factors others than shifting climates.\textsuperscript{51}
Land use changes could significantly decrease bat suitability, notably
through effects on diet and availability of
habitats.\textsuperscript{52} As our results establish that the
diversification of bats betacoronaviruses happens on top of processes
affecting hosts, biogeographic variation in human population density and
anthropogenic disturbances may feed into co-evolutionary dynamics.
Increase in humans-hosts contacts also increase the risk of emergence of
novel diseases,\textsuperscript{53} so does the changes in landscape
connectivity at local/regional scales.\textsuperscript{54} This
represents a challenge for both conservation strategies and disease
ecology: some areas can a high emergence risk and more potential for the
acquisition of zoonotic viruses through bat-human
encounters.\textsuperscript{55} In particular, the challenge ahead lies
in the need to quantify actual exposure (and risk) accounting for
several transmission scenarios, including both direct and indirect bat -
human interactions, and feeding back into the provision of ecosystem
services by bats.

\textbf{Acknowledgements}: We acknowledge that this study was conducted
on land within the traditional unceded territory of the Saint Lawrence
Iroquoian, Anishinabewaki, Mohawk, Huron-Wendat, and Omàmiwininiwak
nations. This work was supported by funding to the Viral Emergence
Research Initiative (VERENA) consortium including NSF BII 2021909 and a
grant from Institut de Valorisation des Données (IVADO). This research
was enabled in part by support provided by Calcul Québec
(www.calculquebec.ca) and Compute Canada (www.computecanada.ca). NF is
funded by the NSERC BIOS² CREATE program. TP and NF are funded by the
Courtois Foundation. RLM was supported by Bryce Carmine and Anne Carmine
(née Percival), through the Massey University Foundation. DJB was
supported by the National Institute of General Medical Sciences of the
National Institutes of Health (P20GM134973).

\newpage

\hypertarget{methods}{%
\section{Methods}\label{methods}}

\hypertarget{known-betacoronavirus-hosts}{%
\subsection{\texorpdfstring{Known \emph{Betacoronavirus}
hosts}{Known Betacoronavirus hosts}}\label{known-betacoronavirus-hosts}}

We downloaded the data on bats hosts of \emph{Betacoronavirus} from
\texttt{https://www.viralemergence.org/betacov} on
Apr.~2022,\textsuperscript{21} and filtered it to ``known'' hosts
(established before the emergence of SARS-CoV-2) and ``novel'' hosts
(confirmed through sampling and competence assays since the initial data
collection). The original database was assembled by a combination of
data mining and literature surveys, including automated alerts on the
``bats'' and ``coronavirus'' keywords to identify novel empirical
evidence of bats-betacoronaviruses associations; this yielded a total of
126 known hosts, 47 of which were novel hosts.

\hypertarget{bat-occurrences}{%
\subsection{Bat occurrences}\label{bat-occurrences}}

We downloaded the rangemap of every current bat species that was
classified as an empirically documented host of \emph{Betacoronavirus}
from the previous step, according to recent IUCN
data.\textsuperscript{56} The range maps were subsequently rasterized
using the \texttt{rasterize} function from
\texttt{GDAL}\textsuperscript{57} at a resolution of approximately
100kmx100km. For every pixel in the resulting raster where at least one
bat host of \emph{Betacoronavirus} was present, we extract the species
pool (list of all known bat hosts), which was used to calculate the
following risk assessment components: bat phylogenetic diversity, bat
compositional uniqueness, and predicted viral sharing risk.

\hypertarget{bat-phylogenetic-diversity}{%
\subsection{Bat phylogenetic
diversity}\label{bat-phylogenetic-diversity}}

For every pixel, we measured Faith's Phylogenetic
Diversity\textsuperscript{58} based on a recent synthetic tree with
robust time calibration, covering about 6000 mammalian
species.\textsuperscript{59} Faith's PD measures the sum of unique
branches from an arbitrary root to a set of tips, and comparatively
larger values indicate a more phylogenetic diverse species pool. We
measured phylogenetic diversity starting from the root of the entire
tree (and not from Chiroptera); this bears no consequences on the
resulting values, since all branches leading up to Chiroptera are only
counted one per species pool, and (as we explain when describing the
assembly of the composite risk map), all individual risk components are
ranged in {[}0,1{]}. This measure incorporates a richness component,
which we chose not to correct for; the interpretation of the
phylogenetic diversity is therefore a weighted species richness, that
accounts for phylogenetic over/under-dispersal in some places.

\hypertarget{bat-compositional-uniqueness}{%
\subsection{Bat compositional
uniqueness}\label{bat-compositional-uniqueness}}

For every species pool, we measured its Local Contribution to
Beta-Diversity;\textsuperscript{60} LCBD works from a species-data
matrix (traditionally noted as \(\mathbf{Y}\)), where species are rows
and sites are columns, and a value of 1 indicates occurrence. We
extracted the Y matrix assuming that every pixel represents a unique
location, and following best practices\textsuperscript{61} transformed
it using Hellinger's distance to account for unequal bat richness at
different pixels. The correction of raw community data is particularly
important for two reasons: first, it prevents the artifact of richer
sites having higher importance; second, it removes the effect of overall
species richness, which is already incorporated in the phylogenetic
diversity component. High values of LCBD indicate that the pixel has a
community that is on average more dissimilar in species composition than
what is expected knowing the entire matrix, i.e.~a more unique
community. Recent results by\textsuperscript{62} shows that LCBD
measures are robust with regards to spatial scale, and are therefore
applicable at the global scale.

\hypertarget{viral-sharing-between-hosts}{%
\subsection{Viral sharing between
hosts}\label{viral-sharing-between-hosts}}

For all bat hosts of \emph{Betacoronavirus}, we extracted their
predicted viral sharing network, generated from a previously published
generalized additive mixed model of virus sharing by a tensor function
of phylogenetic distance and geographic range overlap across
mammals.\textsuperscript{63} This network stores pairwise values of
viral community similarity. To project viral sharing values into a
single value for every pixel, we averaged the pairwise scores. High
values of the average sharing propensity means that this specific extant
bat assemblage is likely to be proficient at exchanging viruses.

\hypertarget{composite-risk-map}{%
\subsection{Composite risk map}\label{composite-risk-map}}

To visualize the aggregated risk at the global scale, we combine the
three individual risk components (phylogenetic diversity, compositional
uniqueness, and viral sharing) using an additive color
model.\textsuperscript{64} In this approach, every risk component gets
assigned a component in the RGB color model (phylogenetic diversity is
green, compositional uniqueness is red, and viral sharing is blue). In
order to achieve a valid RGB measure, all components are re-scaled to
the {[}0,1{]} interval, so that a pixel with no sharing, no phylogenetic
diversity, and no compositional uniqueness is black, and a pixel with
maximal values for each is white. This additive model conveys both the
intensity of the overall risk, but also the nature of the risk as colors
diverge towards combinations of values for three risk components. Out of
the possible combinations, the most risky in terms or rapid
diversification and spillover potential is high phylogenetic diversity
and low viral sharing,\textsuperscript{65} in that this allows multiple
independent host-virus coevolutionary dynamics to take place in the same
location. In the colorimetric space, this correspond to yellow --
because the HSV space is more amenable to calculations for feature
extraction,\textsuperscript{66} we measured the risk level by
calculating the angular distance of the hue of each pixel to a reference
value of 60 (yellow), and weighted this risk level by the value
component. Specifically, given a pixel with colorimetric coordinates
\((h,s,v)\), its ranged weighted risk value is

\[
v\times\left[1-\frac{\left|\text{atan}\left(\text{cos}(\text{rad}(h)), \text{sin}(\text{rad}(h))\right) - X\right|}{2\pi}\right]\,,
\]

where X is
\(\text{atan}\left(\text{cos}(\text{rad}(60)), \text{sin}(\text{rad}(60))\right)\),
a constant approximately equal to \(0.5235\).

\hypertarget{viral-phylogeography-and-evolutionary-diversification}{%
\subsection{Viral phylogeography and evolutionary
diversification}\label{viral-phylogeography-and-evolutionary-diversification}}

To next represent phylogeography of betacoronaviruses in bats, we
aggregated and analyzed betacoronavirus sequence data. We used the
following query to pull all \emph{Betacoronavirus} sequence data from
the GenBank Nucleotide database except SARS-CoV-2;
(``Betacoronavirus''{[}Organism{]} OR betacoronavirus{[}All Fields{]})
NOT (``Severe acute respiratory syndrome coronavirus 2''{[}Organism{]}
OR sars-cov-2{[}All Fields{]}). We added a single representative
sequence for SARS-CoV-2 and manually curated to remove sequences without
the RNA-dependent RNA polymerase (RdRp) sequence or that contained words
indicating recombinant or laboratory strains including ``patent'',
``mutant'', ``GFP'', and ``recombinant''. We filtered over-represented
taxa including betacoronavirus 1, hCoV-OC43, Middle East respiratory
syndrome coronavirus, Murine hepatitis virus, and hCoV-HKU1. Curated
betacoronavirus RdRp sequences were then aligned using
MAFFT\textsuperscript{67} v1.4.0 (Algorithm FFT-NS-2, Scoring matrix
200PAM / k=2, gap open penalty 1.53m offset value 0.123) and a maximum
likelihood tree reconstructed in IQ-TREE\textsuperscript{68} v1.6.12
with ModelFinder\textsuperscript{69} ultrafast bootstrap
approximation\textsuperscript{70} with a general time reversible model
with empirical base frequencies and the 5-discrete-rate-category
FreeRaye model of nucleotide substitution (GTR+F+R5).

We first tested the hypothesis that hotspots of viral diversification
would track hotspots of bat diversification. To do so, we plotted the
number of known bat hosts (specifically only those included in the
phylogeny, so there was a 1:1 correspondence between data sources)
against the ``mean evolutionary distinctiveness'' of the associated
viruses. To calculate this, we derived the fair proportions evolutionary
distinctiveness\textsuperscript{71} for each of the viruses in the tree,
then averaged these at the bat species level, projected these values
onto their geographic distributions, and averaged across every bat found
in a given pixel. As such, this can be thought of as a map of the mean
evolutionary distinctiveness of the known viral community believed to be
associated with a particular subset of bats present.

\hypertarget{co-distribution-of-hosts-and-viral-hotspots}{%
\subsection{Co-distribution of hosts and viral
hotspots}\label{co-distribution-of-hosts-and-viral-hotspots}}

Subsequently, we tested the hypothesis that the biogeography of bat
betacoronaviruses should track the biogeography of their hosts. To test
this idea, we loosely adapted a method from,\textsuperscript{72,73} who
proposed a phylogenetic method for the delineation of animal
biogeographic regions. In their original method, a distance matrix -
where each row or column represents a geographic raster's grid cell, and
the dissimilarity values are the ``beta diversity similarity'' of their
community assemble - undergoes non-metric multidimensional scaling
(NMDS); the first two axes of the NMDS are projected geographically
using a four-color bivariate map. Here, we build on this idea with an
entirely novel methodology. First, we measure the phylogenetic distance
between the different viruses in the betacoronaviruses tree by using the
cophenetic function in \texttt{ape};\textsuperscript{74} subsequently,
we take a principal components analysis of that distance matrix (readily
interchangeable for NMDS in this case) to project the viral tree into an
n-dimensional space. We then take the first two principal components
and, as with the evolutionary distinctiveness analysis, aggregated these
to a mean host value and projected them using a four-color bivariate
map.

\newpage

\hypertarget{references}{%
\section*{References}\label{references}}
\addcontentsline{toc}{section}{References}

\hypertarget{refs}{}
\begin{CSLReferences}{0}{0}
\leavevmode\hypertarget{ref-Plowright2017PatZoo}{}%
\CSLLeftMargin{1. }
\CSLRightInline{Plowright, R. K. \emph{et al.} Pathways to zoonotic
spillover. \emph{Nature Reviews Microbiology} \textbf{15}, 502--510
(2017).}

\leavevmode\hypertarget{ref-Anthony2017GloPat}{}%
\CSLLeftMargin{2. }
\CSLRightInline{Anthony, S. J. \emph{et al.} Global patterns in
coronavirus diversity. \emph{Virus Evolution} \textbf{3}, (2017).}

\leavevmode\hypertarget{ref-Ruiz-Aravena2022EcoEvo}{}%
\CSLLeftMargin{3. }
\CSLRightInline{Ruiz-Aravena, M. \emph{et al.} Ecology, evolution and
spillover of coronaviruses from bats. \emph{Nature Reviews Microbiology}
\textbf{20}, 299--314 (2022).}

\leavevmode\hypertarget{ref-Agosta2010HowSpe}{}%
\CSLLeftMargin{4. }
\CSLRightInline{Agosta, S. J., Janz, N. \& Brooks, D. R. How specialists
can be generalists: Resolving the {`parasite paradox'} and implications
for emerging infectious disease. \emph{Zoologia (Curitiba)} \textbf{27},
151--162 (2010).}

\leavevmode\hypertarget{ref-Power2004PatSpi}{}%
\CSLLeftMargin{5. }
\CSLRightInline{Power, A. G. \& Mitchell, C. E. Pathogen Spillover in
Disease Epidemics. \emph{The American Naturalist} \textbf{164}, S79--S89
(2004).}

\leavevmode\hypertarget{ref-Geoghegan2017ComAna}{}%
\CSLLeftMargin{6. }
\CSLRightInline{Geoghegan, J. L., Duchêne, S. \& Holmes, E. C.
Comparative analysis estimates the relative frequencies of co-divergence
and cross-species transmission within viral families. \emph{PLOS
Pathogens} \textbf{13}, e1006215 (2017).}

\leavevmode\hypertarget{ref-Thompson2005GeoMos}{}%
\CSLLeftMargin{7. }
\CSLRightInline{Thompson, J. N. \emph{The Geographic Mosaic of
Coevolution}. (University Of Chicago Press, 2005).}

\leavevmode\hypertarget{ref-Thompson1994CoePro}{}%
\CSLLeftMargin{8. }
\CSLRightInline{Thompson, J. N. \emph{The Coevolutionary Process}.
(University of Chicago Press, 1994).}

\leavevmode\hypertarget{ref-Janzen1980WheIt}{}%
\CSLLeftMargin{9. }
\CSLRightInline{Janzen, D. H. When is it Coevolution? \emph{Evolution}
\textbf{34}, 611--612 (1980).}

\leavevmode\hypertarget{ref-Price2002MacThe}{}%
\CSLLeftMargin{10. }
\CSLRightInline{Price, P. W. \emph{Macroevolutionary Theory on
Macroecological Patterns}. (Cambridge University Press, 2002).
doi:\href{https://doi.org/10.1017/CBO9780511615030}{10.1017/CBO9780511615030}.}

\leavevmode\hypertarget{ref-Gomulkiewicz2007DosDon}{}%
\CSLLeftMargin{11. }
\CSLRightInline{Gomulkiewicz, R. \emph{et al.} Dos and don'ts of testing
the geographic mosaic theory of coevolution. \emph{Heredity}
\textbf{98}, 249--258 (2007).}

\leavevmode\hypertarget{ref-Thrall2007CoeSym}{}%
\CSLLeftMargin{12. }
\CSLRightInline{Thrall, P. H., Hochberg, M. E., Burdon, J. J. \& Bever,
J. D. Coevolution of symbiotic mutualists and parasites in a community
context. \emph{Trends in Ecology \& Evolution} \textbf{22}, 120--126
(2007).}

\leavevmode\hypertarget{ref-Drexler2014EcoEvo}{}%
\CSLLeftMargin{13. }
\CSLRightInline{Drexler, J. F., Corman, V. M. \& Drosten, C. Ecology,
evolution and classification of bat coronaviruses in the aftermath of
SARS. \emph{Antiviral Research} \textbf{101}, 45--56 (2014).}

\leavevmode\hypertarget{ref-Calisher2006BatImp}{}%
\CSLLeftMargin{14. }
\CSLRightInline{Calisher, C. H., Childs, J. E., Field, H. E., Holmes, K.
V. \& Schountz, T. Bats: Important Reservoir Hosts of Emerging Viruses.
\emph{Clinical Microbiology Reviews} \textbf{19}, 531--545 (2006).}

\leavevmode\hypertarget{ref-Moratelli2015BatZoo}{}%
\CSLLeftMargin{15. }
\CSLRightInline{Moratelli, R. \& Calisher, C. H. Bats and zoonotic
viruses: Can we confidently link bats with emerging deadly viruses?
\emph{Memórias do Instituto Oswaldo Cruz} \textbf{110}, 1--22 (2015).}

\leavevmode\hypertarget{ref-Mollentze2020VirZoo}{}%
\CSLLeftMargin{16. }
\CSLRightInline{Mollentze, N. \& Streicker, D. G. Viral zoonotic risk is
homogenous among taxonomic orders of mammalian and avian reservoir
hosts. \emph{Proceedings of the National Academy of Sciences}
\textbf{117}, 9423--9430 (2020).}

\leavevmode\hypertarget{ref-Carlson2022CliCha}{}%
\CSLLeftMargin{17. }
\CSLRightInline{Carlson, C. J. \emph{et al.} Climate change increases
cross-species viral transmission risk. \emph{Nature} 1--1 (2022)
doi:\href{https://doi.org/10.1038/s41586-022-04788-w}{10.1038/s41586-022-04788-w}.}

\leavevmode\hypertarget{ref-Springer2013PhyBat}{}%
\CSLLeftMargin{18. }
\CSLRightInline{Springer, M. S. Phylogenetics: Bats United, Microbats
Divided. \emph{Current Biology} \textbf{23}, R999--R1001 (2013).}

\leavevmode\hypertarget{ref-Teeling2005MolPhy}{}%
\CSLLeftMargin{19. }
\CSLRightInline{Teeling, E. C. \emph{et al.} A Molecular Phylogeny for
Bats Illuminates Biogeography and the Fossil Record. \emph{Science (New
York, N.Y.)} \textbf{307}, 580--584 (2005).}

\leavevmode\hypertarget{ref-Villalobos2010DivFie}{}%
\CSLLeftMargin{20. }
\CSLRightInline{Villalobos, F. \& Arita, H. T. The diversity field of
New World leaf-nosed bats (Phyllostomidae). \emph{Global Ecology and
Biogeography} \textbf{19}, 200--211 (2010).}

\leavevmode\hypertarget{ref-Becker2022OptPre}{}%
\CSLLeftMargin{21. }
\CSLRightInline{Becker, D. J. \emph{et al.} Optimising predictive models
to prioritise viral discovery in zoonotic reservoirs. \emph{The Lancet
Microbe} (2022)
doi:\href{https://doi.org/10.1016/S2666-5247(21)00245-7}{10.1016/S2666-5247(21)00245-7}.}

\leavevmode\hypertarget{ref-Allen2017GloHot}{}%
\CSLLeftMargin{22. }
\CSLRightInline{Allen, T. \emph{et al.} Global hotspots and correlates
of emerging zoonotic diseases. \emph{Nature Communications} \textbf{8},
(2017).}

\leavevmode\hypertarget{ref-Olival2017HosVir}{}%
\CSLLeftMargin{23. }
\CSLRightInline{Olival, K. J. \emph{et al.} Host and viral traits
predict zoonotic spillover from mammals. \emph{Nature} \textbf{546},
646--650 (2017).}

\leavevmode\hypertarget{ref-VanBrussel2022ZooDis}{}%
\CSLLeftMargin{24. }
\CSLRightInline{Van Brussel, K. \& Holmes, E. C. Zoonotic disease and
virome diversity in bats. \emph{Current Opinion in Virology}
\textbf{52}, 192--202 (2022).}

\leavevmode\hypertarget{ref-Letko2020BatVir}{}%
\CSLLeftMargin{25. }
\CSLRightInline{Letko, M., Seifert, S. N., Olival, K. J., Plowright, R.
K. \& Munster, V. J. Bat-borne virus diversity, spillover and emergence.
\emph{Nature Reviews Microbiology} \textbf{18}, 461--471 (2020).}

\leavevmode\hypertarget{ref-Banerjee2020NovIns}{}%
\CSLLeftMargin{26. }
\CSLRightInline{Banerjee, A. \emph{et al.} Novel Insights Into Immune
Systems of Bats. \emph{Frontiers in Immunology} \textbf{11}, 26 (2020).}

\leavevmode\hypertarget{ref-Gorbunova2020WorGoe}{}%
\CSLLeftMargin{27. }
\CSLRightInline{Gorbunova, V., Seluanov, A. \& Kennedy, B. K. The World
Goes Bats: Living Longer and Tolerating Viruses. \emph{Cell Metabolism}
\textbf{32}, 31--43 (2020).}

\leavevmode\hypertarget{ref-Irving2021LesHos}{}%
\CSLLeftMargin{28. }
\CSLRightInline{Irving, A. T., Ahn, M., Goh, G., Anderson, D. E. \&
Wang, L.-F. Lessons from the host defences of bats, a unique viral
reservoir. \emph{Nature} \textbf{589}, 363--370 (2021).}

\leavevmode\hypertarget{ref-Jebb2020SixRef}{}%
\CSLLeftMargin{29. }
\CSLRightInline{Jebb, D. \emph{et al.} Six reference-quality genomes
reveal evolution of bat adaptations. \emph{Nature} \textbf{583},
578--584 (2020).}

\leavevmode\hypertarget{ref-MorenoSantillan2021LarGen}{}%
\CSLLeftMargin{30. }
\CSLRightInline{Moreno Santillán, D. D. \emph{et al.} Large-scale genome
sampling reveals unique immunity and metabolic adaptations in bats.
\emph{Molecular Ecology} mec.16027 (2021)
doi:\href{https://doi.org/10.1111/mec.16027}{10.1111/mec.16027}.}

\leavevmode\hypertarget{ref-Muylaert2022PreFut}{}%
\CSLLeftMargin{31. }
\CSLRightInline{Muylaert, R. L. \emph{et al.} Present and future
distribution of bat hosts of sarbecoviruses: Implications for
conservation and public health. \emph{Proceedings of the Royal Society
B: Biological Sciences} \textbf{289}, 20220397 (2022).}

\leavevmode\hypertarget{ref-Wang2018SerEvi}{}%
\CSLLeftMargin{32. }
\CSLRightInline{Wang, N. \emph{et al.} Serological Evidence of Bat
SARS-Related Coronavirus Infection in Humans, China. \emph{Virologica
Sinica} \textbf{33}, 104--107 (2018).}

\leavevmode\hypertarget{ref-Ramshaw2019DatGeo}{}%
\CSLLeftMargin{33. }
\CSLRightInline{Ramshaw, R. E. \emph{et al.} A database of geopositioned
Middle East Respiratory Syndrome Coronavirus occurrences.
\emph{Scientific Data} \textbf{6}, 318 (2019).}

\leavevmode\hypertarget{ref-Lednicky2021IsoNov}{}%
\CSLLeftMargin{34. }
\CSLRightInline{Lednicky, J. A. \emph{et al.} Isolation of a Novel
Recombinant Canine Coronavirus from a Visitor to Haiti: Further Evidence
of Transmission of Coronaviruses of Zoonotic Origin to Humans.
\emph{Clinical Infectious Diseases: An Official Publication of the
Infectious Diseases Society of America} ciab924 (2021)
doi:\href{https://doi.org/10.1093/cid/ciab924}{10.1093/cid/ciab924}.}

\leavevmode\hypertarget{ref-Vlasova2022AniAlp}{}%
\CSLLeftMargin{35. }
\CSLRightInline{Vlasova, A. N. \emph{et al.} Animal alphacoronaviruses
found in human patients with acute respiratory illness in different
countries. \emph{Emerging Microbes \& Infections} \textbf{11}, 699--702
(2022).}

\leavevmode\hypertarget{ref-Shi2014DeeDiv}{}%
\CSLLeftMargin{36. }
\CSLRightInline{Shi, J. J. \emph{et al.} A Deep Divergence Time between
Sister Species of Eidolon (Pteropodidae) with Evidence for Widespread
Panmixia. \emph{Acta Chiropterologica} \textbf{16}, 279--292 (2014).}

\leavevmode\hypertarget{ref-Kettenburg2022FulGen}{}%
\CSLLeftMargin{37. }
\CSLRightInline{Kettenburg, G. \emph{et al.} Full Genome Nobecovirus
Sequences From Malagasy Fruit Bats Define a Unique Evolutionary History
for This Coronavirus Clade. \emph{Frontiers in Public Health}
\textbf{10}, (2022).}

\leavevmode\hypertarget{ref-Hazarie2021IntPop}{}%
\CSLLeftMargin{38. }
\CSLRightInline{Hazarie, S., Soriano-Paños, D., Arenas, A.,
Gómez-Gardeñes, J. \& Ghoshal, G. Interplay between population density
and mobility in determining the spread of epidemics in cities.
\emph{Communications Physics} \textbf{4}, 191 (2021).}

\leavevmode\hypertarget{ref-Xu2004EpiClu}{}%
\CSLLeftMargin{39. }
\CSLRightInline{Xu, R.-H. \emph{et al.} Epidemiologic Clues to SARS
Origin in China. \emph{Emerging Infectious Diseases} \textbf{10},
1030--1037 (2004).}

\leavevmode\hypertarget{ref-Cohen2022SamStr}{}%
\CSLLeftMargin{40. }
\CSLRightInline{Cohen, L. E., Fagre, A. C., Chen, B., Carlson, C. J. \&
Becker, D. J. Sampling strategies and pre-pandemic surveillance gaps for
bat coronaviruses. 2022.06.15.496296 (2022)
doi:\href{https://doi.org/10.1101/2022.06.15.496296}{10.1101/2022.06.15.496296}.}

\leavevmode\hypertarget{ref-Peixoto2018SynEco}{}%
\CSLLeftMargin{41. }
\CSLRightInline{Peixoto, F. P., Braga, P. H. P. \& Mendes, P. A
synthesis of ecological and evolutionary determinants of bat diversity
across spatial scales. \emph{BMC ecology} \textbf{18}, 18 (2018).}

\leavevmode\hypertarget{ref-Simmons2020BatSpe}{}%
\CSLLeftMargin{42. }
\CSLRightInline{Simmons, N. B. \& Cirranello, A. L. Bat Species of the
World: A taxonomic and geographic database. \url{https://batnames.org/}
(2020).}

\leavevmode\hypertarget{ref-Kasso2013EcoEco}{}%
\CSLLeftMargin{43. }
\CSLRightInline{Kasso, M. \& Balakrishnan, M. Ecological and Economic
Importance of Bats (Order Chiroptera). \emph{ISRN Biodiversity}
\textbf{2013}, e187415 (2013).}

\leavevmode\hypertarget{ref-Mello2011MisPar}{}%
\CSLLeftMargin{44. }
\CSLRightInline{Mello, M. A. R. \emph{et al.} The Missing Part of Seed
Dispersal Networks: Structure and Robustness of Bat-Fruit Interactions.
\emph{PLOS ONE} \textbf{6}, e17395 (2011).}

\leavevmode\hypertarget{ref-Voigt2016BatAnt}{}%
\CSLLeftMargin{45. }
\CSLRightInline{\emph{Bats in the Anthropocene: Conservation of Bats in
a Changing World}. (Springer International Publishing, 2016).
doi:\href{https://doi.org/10.1007/978-3-319-25220-9}{10.1007/978-3-319-25220-9}.}

\leavevmode\hypertarget{ref-Williams-Guillen2008BatLim}{}%
\CSLLeftMargin{46. }
\CSLRightInline{Williams-Guillén, K., Perfecto, I. \& Vandermeer, J.
Bats Limit Insects in a Neotropical Agroforestry System. \emph{Science}
\textbf{320}, 70--70 (2008).}

\leavevmode\hypertarget{ref-Gonsalves2013MosCon}{}%
\CSLLeftMargin{47. }
\CSLRightInline{Gonsalves, L., Bicknell, B., Law, B., Webb, C. \&
Monamy, V. Mosquito Consumption by Insectivorous Bats: Does Size Matter?
\emph{PLOS ONE} \textbf{8}, e77183 (2013).}

\leavevmode\hypertarget{ref-Gonsalves2013MosInf}{}%
\CSLLeftMargin{48. }
\CSLRightInline{Gonsalves, L., Lamb, S., Webb, C., Law, B. \& Monamy, V.
Do mosquitoes influence bat activity in coastal habitats? \emph{Wildlife
Research} \textbf{40}, 10--24 (2013).}

\leavevmode\hypertarget{ref-Latinne2020OriCro}{}%
\CSLLeftMargin{49. }
\CSLRightInline{Latinne, A. \emph{et al.} Origin and cross-species
transmission of bat coronaviruses in China. \emph{bioRxiv: The Preprint
Server for Biology} 2020.05.31.116061 (2020)
doi:\href{https://doi.org/10.1101/2020.05.31.116061}{10.1101/2020.05.31.116061}.}

\leavevmode\hypertarget{ref-Wu2021ComSur}{}%
\CSLLeftMargin{50. }
\CSLRightInline{Wu, Z. \emph{et al.} \emph{A comprehensive survey of bat
sarbecoviruses across China for the origin tracing of SARS-CoV and
SARS-CoV-2}. (2021)
doi:\href{https://doi.org/10.21203/rs.3.rs-885194/v1}{10.21203/rs.3.rs-885194/v1}.}

\leavevmode\hypertarget{ref-Alves2018GeoVar}{}%
\CSLLeftMargin{51. }
\CSLRightInline{Alves, D. M. C. C., Diniz-Filho, J. A. F., da Silva e
Souza, K., Gouveia, S. F. \& Villalobos, F. Geographic variation in the
relationship between large-scale environmental determinants and bat
species richness. \emph{Basic and Applied Ecology} \textbf{27}, 1--8
(2018).}

\leavevmode\hypertarget{ref-Treitler2016EffLoc}{}%
\CSLLeftMargin{52. }
\CSLRightInline{Treitler, J. T., Heim, O., Tschapka, M. \& Jung, K. The
effect of local land use and loss of forests on bats and nocturnal
insects. \emph{Ecology and Evolution} \textbf{6}, 4289--4297 (2016).}

\leavevmode\hypertarget{ref-Johnson2020GloShi}{}%
\CSLLeftMargin{53. }
\CSLRightInline{Johnson, C. K. \emph{et al.} Global shifts in mammalian
population trends reveal key predictors of virus spillover risk.
\emph{Proceedings of the Royal Society B: Biological Sciences}
\textbf{287}, 20192736 (2020).}

\leavevmode\hypertarget{ref-Gryseels2017WheVir}{}%
\CSLLeftMargin{54. }
\CSLRightInline{Gryseels, S. \emph{et al.} When Viruses Don't Go Viral:
The Importance of Host Phylogeographic Structure in the Spatial Spread
of Arenaviruses. \emph{PLOS Pathogens} \textbf{13}, e1006073 (2017).}

\leavevmode\hypertarget{ref-Amman2011InvRol}{}%
\CSLLeftMargin{55. }
\CSLRightInline{Amman, B. R. \emph{et al.} \emph{Investigating the role
of bats in emerging zoonoses: Balancing ecology, conservation and public
health interest}. (FAO, 2011).}

\leavevmode\hypertarget{ref-IUCN2021IucRed}{}%
\CSLLeftMargin{56. }
\CSLRightInline{IUCN. The IUCN Red List of Threatened Species.
\url{https://www.iucnredlist.org} (2021).}

\leavevmode\hypertarget{ref-RouaultEven2022GdaOgr}{}%
\CSLLeftMargin{57. }
\CSLRightInline{Rouault, E. \emph{et al.} \emph{GDAL/OGR Geospatial Data
Abstraction software Library}. (Zenodo, 2022).
doi:\href{https://doi.org/10.5281/ZENODO.5884351}{10.5281/ZENODO.5884351}.}

\leavevmode\hypertarget{ref-Faith1992ConEva}{}%
\CSLLeftMargin{58. }
\CSLRightInline{Faith, D. P. Conservation evaluation and phylogenetic
diversity. \emph{Biological Conservation} \textbf{61}, 1--10 (1992).}

\leavevmode\hypertarget{ref-Upham2019InfMam}{}%
\CSLLeftMargin{59. }
\CSLRightInline{Upham, N. S., Esselstyn, J. A. \& Jetz, W. Inferring the
mammal tree: Species-level sets of phylogenies for questions in ecology,
evolution, and conservation. \emph{PLOS Biology} \textbf{17}, e3000494
(2019).}

\leavevmode\hypertarget{ref-Legendre2013BetDiv}{}%
\CSLLeftMargin{60. }
\CSLRightInline{Legendre, P. \& De Cáceres, M. Beta diversity as the
variance of community data: Dissimilarity coefficients and partitioning.
\emph{Ecology Letters} \textbf{16}, 951--963 (2013).}

\leavevmode\hypertarget{ref-Legendre2019SpaTem}{}%
\CSLLeftMargin{61. }
\CSLRightInline{Legendre, P. \& Condit, R. Spatial and temporal analysis
of beta diversity in the Barro Colorado Island forest dynamics plot,
Panama. \emph{Forest Ecosystems} \textbf{6}, 7 (2019).}

\leavevmode\hypertarget{ref-Dansereau2022EvaEco}{}%
\CSLLeftMargin{62. }
\CSLRightInline{Dansereau, G., Legendre, P. \& Poisot, T. Evaluating
ecological uniqueness over broad spatial extents using species
distribution modelling. \emph{Oikos} \textbf{n/a}, e09063 (2022).}

\leavevmode\hypertarget{ref-Albery2020PreGlo}{}%
\CSLLeftMargin{63. }
\CSLRightInline{Albery, G. F., Eskew, E. A., Ross, N. \& Olival, K. J.
Predicting the global mammalian viral sharing network using
phylogeography. \emph{Nature Communications} \textbf{11}, 2260 (2020).}

\leavevmode\hypertarget{ref-Seekell2018GeoLak}{}%
\CSLLeftMargin{64. }
\CSLRightInline{Seekell, D. A., Lapierre, J.-F. \& Cheruvelil, K. S. A
geography of lake carbon cycling. \emph{Limnology and Oceanography
Letters} \textbf{3}, 49--56 (2018).}

\leavevmode\hypertarget{ref-Gomulkiewicz2000HotSpo}{}%
\CSLLeftMargin{65. }
\CSLRightInline{Gomulkiewicz, R., Thompson, J. N., Holt, R. D., Nuismer,
S. L. \& Hochberg, M. E. Hot Spots, Cold Spots, and the Geographic
Mosaic Theory of Coevolution. \emph{The American Naturalist}
\textbf{156}, 156--174 (2000).}

\leavevmode\hypertarget{ref-Keke2010StuSki}{}%
\CSLLeftMargin{66. }
\CSLRightInline{Keke, S., Peng, Z. \& Guohui, L. Study on skin color
image segmentation used by Fuzzy-c-means arithmetic. in \emph{2010
Seventh International Conference on Fuzzy Systems and Knowledge
Discovery} vol. 2 612--615 (2010).}

\leavevmode\hypertarget{ref-Katoh2013MafMul}{}%
\CSLLeftMargin{67. }
\CSLRightInline{Katoh, K. \& Standley, D. M. MAFFT Multiple Sequence
Alignment Software Version 7: Improvements in Performance and Usability.
\emph{Molecular Biology and Evolution} \textbf{30}, 772--780 (2013).}

\leavevmode\hypertarget{ref-Nguyen2015IqtFas}{}%
\CSLLeftMargin{68. }
\CSLRightInline{Nguyen, L.-T., Schmidt, H. A., von Haeseler, A. \& Minh,
B. Q. IQ-TREE: A Fast and Effective Stochastic Algorithm for Estimating
Maximum-Likelihood Phylogenies. \emph{Molecular Biology and Evolution}
\textbf{32}, 268--274 (2015).}

\leavevmode\hypertarget{ref-Kalyaanamoorthy2017ModFas}{}%
\CSLLeftMargin{69. }
\CSLRightInline{Kalyaanamoorthy, S., Minh, B. Q., Wong, T. K. F., von
Haeseler, A. \& Jermiin, L. S. ModelFinder: Fast model selection for
accurate phylogenetic estimates. \emph{Nature Methods} \textbf{14},
587--589 (2017).}

\leavevmode\hypertarget{ref-Hoang2018UfbImp}{}%
\CSLLeftMargin{70. }
\CSLRightInline{Hoang, D. T., Chernomor, O., von Haeseler, A., Minh, B.
Q. \& Vinh, L. S. UFBoot2: Improving the Ultrafast Bootstrap
Approximation. \emph{Molecular Biology and Evolution} \textbf{35},
518--522 (2018).}

\leavevmode\hypertarget{ref-Isaac2007MamEdg}{}%
\CSLLeftMargin{71. }
\CSLRightInline{Isaac, N. J. B., Turvey, S. T., Collen, B., Waterman, C.
\& Baillie, J. E. M. Mammals on the EDGE: Conservation Priorities Based
on Threat and Phylogeny. \emph{PLOS ONE} \textbf{2}, e296 (2007).}

\leavevmode\hypertarget{ref-Kreft2007GloPat}{}%
\CSLLeftMargin{72. }
\CSLRightInline{Kreft, H. \& Jetz, W. Global patterns and determinants
of vascular plant diversity. \emph{Proceedings of the National Academy
of Sciences} \textbf{104}, 5925--5930 (2007).}

\leavevmode\hypertarget{ref-Kreft2010FraDel}{}%
\CSLLeftMargin{73. }
\CSLRightInline{Kreft, H. \& Jetz, W. A framework for delineating
biogeographical regions based on species distributions. \emph{Journal of
Biogeography} \textbf{37}, 2029--2053 (2010).}

\leavevmode\hypertarget{ref-Paradis2019ApeEnv}{}%
\CSLLeftMargin{74. }
\CSLRightInline{Paradis, E. \& Schliep, K. Ape 5.0: An environment for
modern phylogenetics and evolutionary analyses in R.
\emph{Bioinformatics} \textbf{35}, 526--528 (2019).}

\end{CSLReferences}

\end{document}
